<!DOCTYPE HTML>
<html>
<head><meta name="generator" content="Hexo 3.9.0">
  <meta charset="utf-8">
  
  <title>Page 5 | leesei&#39;s notebook</title>

  <meta name="author" content="leesei">
  
  <meta name="description" content="Technical blog for the future me and other beings.">
  
  
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

  
  <meta property="og:site_name" content="leesei&#39;s notebook">

  
    <meta property="og:image" content>
  

  
  
    <link href="/favicon.png" rel="icon">
  

  <link rel="stylesheet" href="/css/bootstrap.min.css" media="screen" type="text/css">
  <link rel="stylesheet" href="/css/font-awesome.css" media="screen" type="text/css">
  <link rel="stylesheet" href="/css/style.css" media="screen" type="text/css">
  <link rel="stylesheet" href="/css/highlight.css" media="screen" type="text/css">
  <link rel="stylesheet" href="/css/google-fonts.css" media="screen" type="text/css">
  <!--[if lt IE 9]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->

  <script src="/js/jquery-2.0.3.min.js"></script>

  <!-- analytics -->
  



</head>
</html>
 <body>  
  <nav id="main-nav" class="navbar navbar-inverse navbar-fixed-top" role="navigation">
    <div class="container">
      <button type="button" class="navbar-header navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
		<span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
	  <a class="navbar-brand" href="/">leesei&#39;s notebook</a>
      <div class="collapse navbar-collapse nav-menu">
		<ul class="nav navbar-nav">
		  
		  <li>
			<a href="/archives" title="All the articles.">
			  <i class="fa fa-archive"></i>Archives
			</a>
		  </li>
		  
		  <li>
			<a href="/categories" title="All the categories.">
			  <i class="fa fa-folder"></i>Categories
			</a>
		  </li>
		  
		  <li>
			<a href="/tags" title="All the tags.">
			  <i class="fa fa-tags"></i>Tags
			</a>
		  </li>
		  
		  <li>
			<a href="/about" title="About me.">
			  <i class="fa fa-user"></i>About
			</a>
		  </li>
		  
		</ul>
      </div>
    </div> <!-- container -->
</nav>
<div class="clearfix"></div>

  <div class="container">
  	<div class="content">
    	 <div class="page-header">
  <h1>εмβяαςε мλ sμρεя gεεκ ροωεя.</h1>
</div>

<div class="row page">

	
	<div class="col-md-9">
	

		<div class="slogan">
      Technical blog for the future me and other beings.
</div>


		<div class="mypage">
		
		<!-- title and entry -->
		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacids/proteinhydrolysis/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>protein hydrolysis</title>

<meta name="description" content="A brief look at the hydrolysis of proteins using hydrochloric acid">
<meta name="keywords" content="amino, acid, acids, protein, proteins, hydrolysis, hydrolyse">

</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p><b>THE HYDROLYSIS OF PROTEINS</b></p></font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="3">
<p>This page looks briefly at the hydrolysis of proteins into their constituent amino acids using hydrochloric acid.</p>
</font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>Hydrolysing proteins using hydrochloric acid</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">

<p><b>The chemistry of the reaction</b></p>

<p>If you have already studied the hydrolysis of amides under acidic conditions, you will find that this is basically the same reaction.  That's not surprising because what biologists and biochemists call a peptide link (in proteins, for example) is what chemists call an amide link.</p>

<p>With an amide like ethanamide, the carbon-nitrogen bond in the amide group is broken and you get a carboxylic acid formed:</p>

<p><img src="padding.gif" width="40" height="15"><img src="hydrolamide.gif" width="378" height="18"></p>

<p>Now imagine doing the same thing with a simple dipeptide made of any two amino acids.</p>

<p align="center"><img src="hydrolprotein.gif" width="285" height="213"></p>

<p>Instead of ammonium ions, you get positive ions made from the -NH<sub>2</sub> groups reacting with hydrogen ions.</p>

<p>You need the extra hydrogen ion in the equation (compared with the amide equation) to react with the -NH<sub>2</sub> group on the left-hand end of the dipeptide - the one not involved in the peptide link.</p>

<p>If you scale this up to a polypeptide (a protein chain), each of the peptide links will be broken in exactly the same way.  That means that you will end up with a mixture of the amino acids that made up the protein - although in the form of their positive ions because of the presence of the hydrogen ions from the hydrochloric acid.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you aren't sure about the formation of these positive ions from the reactions between <a href="acidbase.html">amino acids and hydrogen ions</a>, it would be useful to follow this link.</p>

<p>Use the BACK button on your browser to return to this page.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">

<p><b>Doing the reaction</b></p>

<p>There are two ways of carrying out this reaction - an old, slow method, and a new, fast one.</p>

<p><b><i>The old slow way</i></b></p>

<p>The protein is heated with 6 M hydrochloric acid for about 24 hours at 110&deg;C.  (6M hydrochloric acid is slightly more than semi-concentrated.)</p>

<p><b><i>The new fast way</i></b></p>

<p>Protein samples are placed in tubes in a sealed container containing 6 M hydrochloric acid in an atmosphere of nitrogen.</p>

<p>The whole container is then placed in a microwave oven for about 5 - 30 minutes (depending on the protein) with temperatures up to 200&deg;C.</p>

<p>The hydrochloric acid vaporises, comes into contact with the protein samples and hydrolyses them.</p>

<p>This method is used to hydrolyse small samples of protein during protein analysis.</p>
</font>
<pre>

</pre>
<table align="center" width="100%" cellpadding="5" border="1">
<tr><td bgcolor="#ccffcc"><font color="#000000" face="Helvetica, Arial" size="2">
<p><b>Questions to test your understanding</b></p>

<p>If this is the first set of questions you have done, please read the <a href="../../questions.html">introductory page</a> before you start.  You will need to use the BACK BUTTON on your browser to come back here afterwards.</p>

<p><a href="../questions/q-proteinhydrolysis.pdf">questions on the hydrolysis of proteins</a></p>
<p><a href="../questions/a-proteinhydrolysis.pdf">answers</a></p>

</font>
</td></tr>
</table>
<pre>

</pre>
<dl>
<font color="#006600" face="Arial, Helvetica " size="3">
<dt><p><b>Where would you like to go now?</b></p></dt>

<dd><p><a href="../aminoacidmenu.html#top"><b>To the amino acid and proteins menu . . .</b></a></p>
<p><a href="../../orgpropsmenu.html#top"><b>To the menu of other organic compounds . . .</b></a></p>
<p><a href="../../index.html#top"><b>To Main Menu . . .</b></a></p></dd>
</font></dl>

<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2004 (modified May 2016)</font></p>
</td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacids/proteinhydrolysis/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacids/enzymes3/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>enzyme inhibitors</title>

<meta name="description" content="A simple explanation of competitive and non-competitive enzyme inhibitors">
<meta name="keywords" content="protein, proteins, structure, structures, enzyme, enzymes, catalyst, catalysis, active site, inhibit, inhibition, inhibitor, inhibitors, competitive, non-competitive">

</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p><b>ENZYME INHIBITORS</b></p></font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="3">
<p>This page looks at the effect of inhibitors on reactions involving enzymes.  This is the third and final page talking about how enzymes function as catalysts.  Please remember that this series of pages is written for 16 - 18 year old <i>chemistry</i> students.  If you want something more advanced, you are looking in the wrong place.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>This page assumes that you have already read the page about <a href="enzymes.html#top">how proteins function as enzymes</a>.  If you have come straight to this page via a search engine, you really ought to go back and read that page first.  In fact, unless you already have a good knowledge of protein structure, you may have to go back further still to a page about <a href="proteinstruct.html#top">the structure of proteins</a>.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>Competitive and non-competitive inhibition</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p><b>Competitive inhibitors</b></p>

<p>This is the most straightforward and obvious form of enzyme inhibition - and the name tells you exactly what happens.</p>

<p>The inhibitor has a similar shape to the usual substrate for the enzyme, and competes with it for the active site.  However, once it is attached to the active site, nothing happens to it.  It doesn't react - essentially, it just gets in the way.</p>

<p>Remember the general equation for an enzyme reacting with a substrate?</p>
<pre>

</pre>
<p align="center"><img src="summaryeqn.gif" width="405" height="15"></p>
<pre>

</pre>
<p>The equivalent equation for a competitive inhibitor looks like this:</p>
<pre>

</pre>
<p align="center"><img src="compinhibeqn.gif" width="254" height="18"></p>
<pre>

</pre>
<p>The complex doesn't react any further to form products - but its formation is still <i>reversible</i>.  It breaks up again to form the enzyme and the inhibitor molecule.</p>

<p>That means that if you increase the concentration of the substrate, the substrate can out-compete the inhibitor, and so the normal reaction can take place at a reasonable rate.</p>

<p>A simple example of this involves malonate ions inhibiting the enzyme succinate dehydrogenase.  This enzyme catalyses the conversion of succinate ions to fumarate ions.  The modern names are:</p>

<ul>
<li><p>malonate:  propanedioate</p></li>

<li><p>succinate:  butanedioate</p></li>

<li><p>fumarate:  trans-butenedioate</p></li>
</ul>

<p>The conversion that succinate dehydrogenase carries out is:</p>

<p align="center"><img src="succdh.gif" width="246" height="127"></p>

<p>The reaction is inhibited by malonate ions which have a very similar shape to succinate ions.</p>

<p align="center"><img src="malonate.gif" width="48" height="105"></p>

<p>The similar shape lets the malonate ions bind to the active site, but the lack of the CH<sub>2</sub>-CH<sub>2</sub> bond in the centre of the ion stops any further reaction taking place.</p>

<p>The malonate ions therefore block the active site - but remember that this is reversible.  The malonate ions will break away and free up the enzyme again.  The malonate ions are in competition for the site - they aren't destroying it.</p>

<p>If the succinate ions have a greater concentration than the malonate ions, by chance they will get access to the site more often than the malonate ions.  That means that you can overcome the effect of a competitive inhibitor by increasing the concentration of the substrate.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>Some biochemists are very sloppy about the use of chemical  names!  If you read about this reaction in other places, you may find the reaction quoted as converting succinic acid into fumaric acid.  Some sources even seem to imply that the words &quot;succinate&quot; and &quot;succinic acid&quot; mean the same thing.  That's unacceptable from a chemistry point of view!</p>

<p>As far as I have been able to discover, the correct version is the conversion from succinate ions to fumarate ions (as above).  At a typical cell pH of around pH 7, these substances will be present mainly as their ions, not as the unionised acids.  And scientific papers from people working on the mechanisms for this conversion all talk about the ions, not the acids.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">
<p><b>Non-competitive inhibitors</b></p>

<p>A non-competitive inhibitor doesn't attach itself to the active site, but attaches somewhere else on the enzyme.  By attaching somewhere else it affects the structure of the enzyme and so the way the enzyme works.  Because there isn't any competition involved between the inhibitor and the substrate, increasing the substrate concentration won't help.</p>

<p>If you look at various biochemistry sites on the web, you will find two explanations for this.  We'll look at the simple, fairly obvious one in some detail in a minute.  I want to have a brief word about the other one first.</p>

<p><b><i>&quot;Pure&quot; non-competitive inhibitors</i></b></p>

<p>This explanation says that the inhibitor doesn't affect the ability of the substrate to bond with the active site, but stops it reacting once it is there.</p>

<p>I found a couple of biochemistry sites which said that inhibitors working like this (which they describe as <i>pure</i> non-competitive inhibitors) are virtually unknown.  As a non-biochemist, I don't know what the truth is about this - if you want to find out, you will probably have to do a biochemistry degree!</p>
<pre>

</pre>
<p><b><i>Other non-competitive inhibitors</i></b></p>

<p>The straightforward explanation (which would seem to apply to most enzymes) is that reaction with the inhibitor causes the shape of the active site to change.  Remember that non-competitive inhibitors aren't attaching directly to the active site, but elsewhere on the enzyme.</p>

<p>The inhibitor attachs to a side group in the protein chain, and affects the way the protein folds into its tertiary structure.  That in turn changes the shape of the active site.  If the shape of the active site changes, then the substrate can't attach to it any more.</p>

<p>Some non-competitive inhibitors attach irreversibly to the enzyme, and therefore stop it working permanently.  Others attach reversibly.</p>

<p>A relatively uncomplicated example of non-competitive inhibitors in a reasonably familiar situation is:</p>
<pre>

</pre>
<p><i>Heavy metal poisoning</i></p>

<p>You are probably aware that compounds containing heavy metals such as lead, mercury, copper or silver are poisonous.  This is because ions of these metals are non-competitive inhibitors for several enzymes.</p>

<p>I'm going to take silver as a simple example.</p>

<p>Silver ions react with -SH groups in the side groups of cysteine residues in the protein chain:</p>

<p align="center"><img src="silverinhib.gif" width="394" height="109"></p>

<p>There isn't enough electronegativity difference between silver and sulphur for a full ionic bond and so the bond can be considered as covalent.</p>

<p>If the cysteine residue is somewhere on the protein chain which affects the way it folds into its tertiary structure, then altering this group could have an effect on the shape of the active site, and so stop the enzyme from working.</p>

<p>The 2+ ions from, for example, mercury, copper or lead can behave similarly - also attaching themselves to the sulphur in place of the hydrogen.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>I have come across a couple of references to the ability of heavy metal ions to break sulphur-sulphur bridges, which would have a major effect on the folding of the protein chain.</p>

<p>A student suggested this page, which has a <a href="http://www.wissensdrang.com/auf1hg.htm" target="_blank" rel="noopener">note about mercury poisoning</a> (see item 4 on the page).  This suggests that mercury breaks sulphur-sulphur bridges by attaching to both of the sulphur atoms, distorting the bridge, and so the folding of the protein chains.  I have no reason to suppose that this description is wrong, but I am a bit wary of relying on a single source.</p>

<p>The reason for choosing silver as my example rather than the more obvious mercury or lead is that it forms a 1+ ion.  For 2+ ions, you would have to worry about what got attached to the metal as well as the sulphur - or whether it retained a single positive charge.  All the biochemistry sources I've been able to find skip over this problem - as a chemistry teacher looking for chemical accuracy, I'm not prepared to do that!<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<pre>

</pre>
<table align="center" width="100%" cellpadding="5" border="1">
<tr><td bgcolor="#ccffcc"><font color="#000000" face="Helvetica, Arial" size="2">
<p><b>Questions to test your understanding</b></p>

<p>If this is the first set of questions you have done, please read the <a href="../../questions.html">introductory page</a> before you start.  You will need to use the BACK BUTTON on your browser to come back here afterwards.</p>

<p><a href="../questions/q-proteinenz3.pdf">questions on enzyme inibitors</a></p>
<p><a href="../questions/a-proteinenz3.pdf">answers</a></p>

</font>
</td></tr>
</table>
<pre>

</pre>
<dl>
<font color="#006600" face="Arial, Helvetica " size="3">
<dt><p><b>Where would you like to go now?</b></p></dt>

<dd><p><a href="../aminoacidmenu.html#top"><b>To the amino acid and proteins menu . . .</b></a></p>
<p><a href="../../orgpropsmenu.html#top"><b>To the menu of other organic compounds . . .</b></a></p>
<p><a href="../../index.html#top"><b>To Main Menu . . .</b></a></p></dd>
</font></dl>

<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2007 (last modified May 2016)</font></p>
</td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacids/enzymes3/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacids/enzymes2/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>the effect of changing conditions in enzyme catalysis</title>

<meta name="description" content="An explanation of the effect of substrate concentration, temperature and pH on enzymes">
<meta name="keywords" content="protein, proteins, structure, structures, enzyme, enzymes, catalyst, catalysis, active site, concentration, temperature, ph, denature">

</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p><b>THE EFFECT OF CHANGING CONDITIONS IN ENZYME CATALYSIS</b></p></font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="3">
<p>This page looks at the effect of changing substrate concentration, temperature and pH on reactions involving enzymes.  It follows on from a page describing in simple terms how enzymes function as catalysts.  Please remember that this series of pages is written for 16 - 18 year old <i>chemistry</i> students.  If you want something more advanced, you are looking in the wrong place.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>This page assumes that you have already read the page about <a href="enzymes.html#top">how proteins function as enzymes</a>.  If you have come straight to this page via a search engine, you really ought to go back and read that page first.  In fact, unless you already have a good knowledge of protein structure, you may have to go back further still to a page about <a href="proteinstruct.html#top">the structure of proteins</a>.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>The effect of substrate concentration on the rate of enzyme-controlled reactions</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p>Remember that in biology or biochemistry, the reactant in an enzyme reaction is known as the &quot;substrate&quot;.</p>

<p>What follows is a very brief and simple look at a very complicated topic.  Anything beyond this is the stuff of biochemistry degree courses!</p>
<pre>

</pre>
<p><b>A reminder about the effect of concentration on rate in ordinary chemical reactions</b></p>

<p>If you have done any work on rates of reaction (especially if you have done orders of reaction), you will have come across cases where the rate of reaction is proportional to the concentration of a reactant, or perhaps to the square of its concentration.</p>

<p>You would discover this by changing the concentration of one of the reactants, keeping everything else constant, and measuring the initial rate of the reaction.  If you measure the rate after the reaction has been going for a while, the concentration of the reactant(s) will have changed and that just complicates things.  That's why initial rates are so useful - you know exactly how much you have of everything.</p>

<p>If you plotted a graph of initial reaction rate against the concentration of a reactant, then there are various possibilities depending on the relationship between the concentration and the rate.</p>

<p><b><i>If the rate is independent of the concentration</i></b></p>

<p>This is called a zero order reaction.</p>

<p align="center"><img src="order0.gif" width="328" height="197"></p>
<pre>

</pre>
<p><b><i>If the rate is proportional to the concentration</i></b></p>

<p>This is called a first order reaction.</p>

<p align="center"><img src="order1.gif" width="333" height="197"></p>
<pre>

</pre>
<p><b><i>If the rate is proportional to some power of the concentration greater than one</i></b></p>

<p>In this case, you get a curve.  If the rate was proportional to the square of the concentration, that's a second order reaction.</p>

<p align="center"><img src="orderx.gif" width="315" height="203"></p>
<pre>

</pre>
<p>With reactions controlled by enzymes, you get a completely different type of graph.</p>
<pre>

</pre>
<p><b>Plotting initial rates of enzyme-controlled reactions against substrate concentration</b></p>

<p>The graph for enzyme controlled reactions looks like this:</p>

<p align="center"><img src="enzymerate1.gif" width="276" height="198"></p>
<pre>

</pre>
<p>Two minor things to notice before we discuss it . . .</p>

<ul>
<li><p>Biochemists talk about a reaction velocity instead of a reaction rate.  If you have done any physics, you will know that this is a complete misuse of the word &quot;velocity&quot;!  But that's what you will find in biochemistry sources, so that's what we will have to use.</p></li>

<li><p>In chemistry, rates are normally measured in terms of rate of change of concentration, with units like mol dm<sup>-3</sup> s<sup>-1</sup> (moles per cubic decimetre per second).  Biochemists often quote it in terms of the number of molecules of substrate which a single molecule of enzyme is processing per unit time - per second, for example.  It is easier to visualise, but involves a messy calculation to get there.  That's not our problem for this topic!</p></li><p></p>
</ul>

<p>So why is the graph the shape it is?</p>

<p>For very, very low substrate concentrations, the graph is almost a straight line - like the second chemistry rate graph above.  In other words, at very, very low concentrations, the rate is proportional to the substrate concentration.</p>

<p>But as concentration increases, increasing the concentration more has less and less effect - and eventually the rate reaches a maximum.  Increasing the concentration any more makes no difference to the rate of the reaction.</p>

<p>If you know about orders of reaction, the reaction has now become zero order with respect to the substrate.</p>

<p>The reason for this is actually fairly obvious if you think about it.  After a certain concentration of substrate is reached, every enzyme molecule present in the mixture is working as fast as it can.  If you increase the substrate concentration any more, there aren't any enzyme molecules free to help the extra substrate molecules to react.</p>

<p>Is this unique to enzyme-controlled reactions?  No!  It can happen in some ordinary chemistry cases as well, usually involving a solid catalyst working with gases.  At very high gas pressures (in other words, very high concentrations of gas molecules), the surface of the catalyst can be completely full of gas molecules.  If you increase the amount of gas any more, there isn't any available surface for it to stick to and react.</p>
<pre>

</pre>
<p><b>V<sub>max</sub> and K<sub>M</sub></b></p>

<p>The maximum rate for a particular enzyme reaction is known as <b><i>V<sub>max</sub></i></b>.  (That's V for velocity - a bit confusing for chemists where V is almost always used for volume!)</p>

<p>This is easily measured by drawing a line on the graph:</p>

<p align="center"><img src="enzymerate2.gif" width="276" height="198"></p>
<pre>

</pre>
<p>This is sometimes reported as a &quot;turnover number&quot;, measured as the number of molecules of substrate processed by a single enzyme molecule per second, or per minute.</p>

<p><b><i>K<sub>M</sub></i></b> is known as the Michaelis constant or the Michaelis-Menten constant (for reasons which needn't concern us), and is a useful measure of the efficiency of an enzyme.</p>

<p>K<sub>M</sub> is the concentration of the substrate in mol dm<sup>-3</sup> which produces a reaction rate of half V<sub>max</sub>.  So it is found like this . . .</p>

<p align="center"><img src="enzymerate3.gif" width="289" height="228"></p>

<p>A low value of K<sub>M</sub> means that the reaction is going quickly even at low substrate concentrations.  A higher value means the enzyme isn't as effective.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>You might possibly wonder why you have to go to the trouble of halving the maximum rate to get this information.  Couldn't you just find the concentration of the substrate which produced the maximum rate?</p>

<p>If you look at the shape of the graph, it would be impossible to get any accurate measure of the concentration which first produced a maximum rate.  The curve just gradually gets closer and closer to the horizontal, and there is no clear cut-off point at which it suddenly becomes horizontal.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>The effect of temperature on the rate of enzyme-controlled reactions</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">

<p><b>A reminder about the effect of temperature on rate in ordinary chemical reactions</b></p>

<p>Remember that for molecules to react, they have to collide with an energy equal to or greater than the activation energy for the reaction.</p>

<p>Heating a reaction makes the molecules move faster and so collide more often.  More collisions in a given time means a faster reaction.</p>

<p>But far more importantly, increasing the temperature has a very big effect on the number of collisions with enough energy to react.  Quite a small temperature rise can produce a large increase in rate.</p>

<p>As a reasonable approximation for many (although not all) reactions close to room temperature, a 10&deg;C increase in temperature will double the rate of a chemical reaction.  This is only an approximation - it may take 9&deg;C or 11&deg;C or whatever, but it gives you an idea of what to expect.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you aren't sure about any of this, have a quick look at the page about the <a href="../../physical/basicrates/temperature.html#top">effect of temperature on rates of reaction</a>.</p>

<p>Use the BACK button on your browser to come back here afterwards.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p><b>Plotting rates of enzyme-controlled reactions against temperature</b></p>

<p>For low temperatures up to about 40&deg;C, enzyme-controlled reactions behave much as you would expect any other chemical reaction to behave.  But above about 40&deg;C (the exact temperature varies from enzyme to enzyme), there is a dramatic fall in reaction rate.</p>

<p>A typical graph of rate against temperature might look like this:</p>
<pre>

</pre>
<p align="center"><img src="enzymeratet.gif" width="308" height="193"></p>

<p>The temperature at which the rate is fastest is called the <b><i>optimum temperature</i></b> for that enzyme.</p>

<p>Different enzymes have different optimum temperatures.  Some enzymes, for example, in organisms known as thermophiles or extremophiles are capable of working at temperatures like 80&deg;C or even higher.</p>

<p>The optimum temperature for a particular enzyme varies depending on how long it is exposed to the higher temperatures.  Enzymes can withstand temperatures higher than their normal optimum if they are only exposed to the higher temperatures for a very short time.</p>
<pre>

</pre>
<p><b>Explaining the rate against temperature graph</b></p>

<p>At lower temperatures, the shape of the graph is exactly what you would expect for any reaction.  All that needs explaining is why the rate falls above the optimum temperature.</p>

<p>Remember that the enzyme works because its substrate fits into the active site on the protein molecule.  That active site is produced because of the way the protein is folded into its tertiary structure.</p>

<p>Heating an enzyme gives the protein chains extra energy and makes them move more.  If they move enough, then the bonds holding the tertiary structure in place will come under increasing strain.</p>

<p>The weaker bonds will break first - van der Waals attractions between side groups, and then hydrogen bonds.  As soon as these bonds holding the tertiary structure together are broken, then the shape of the active site is likely to be lost.</p>

<p>Obviously, the longer the enzyme is held at the higher temperature, the more time there is for the enzyme structure to be broken up.  Given enough time and high enough temperatures, the enzyme structure is broken permanently.  The enzyme is said to be <b><i>denatured</i></b>.  For example, boiling an egg for 5 minutes denatures the proteins in the egg.  Once that has happened, you can't un-boil it by cooling it!</p>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>The effect of pH on the rate of enzyme-controlled reactions</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p>In the same way that every enzyme has an optimum temperature, so each enzyme also has an optimum pH at which it works best.</p>

<p>For example, trypsin and pepsin are both enzymes in the digestive system which break protein chains in the food into smaller bits - either into smaller peptide chains or into individual amino acids.</p>

<p>Pepsin works in the highly acidic conditions of the stomach.  It has an optimum pH of about 1.5.</p>

<p>On the other hand, trypsin works in the small intestine, parts of which have a pH of around 7.5.  Trypsin's optimum pH is about 8.</p>

<p>If you think about the structure of an enzyme molecule, and the sorts of bonds that it may form with its substrate, it isn't surprising that pH should matter.</p>

<p>Suppose an enzyme has an optimum pH around 7.  Imagine that at a pH of around 7, a substrate attaches itself to the enzyme via two ionic bonds.   In the diagram below, the groups allowing ionic bonding are caused by the transfer of a hydrogen ion from a -COOH group in the side chain of one amino acid residue to an -NH<sub>2</sub> group in the side chain of another.</p>

<p>In this simplified example, that is equally true in both the substrate and the enzyme.</p>

<p align="center"><img src="asiteph1.gif" width="243" height="184"></p>
<pre>

</pre>
<p>Now think about what happens at a lower pH - in other words under acidic conditions.</p>

<p>It won't affect the -NH<sub>3</sub><sup>+</sup> group, but the -COO<sup>-</sup> will pick up a hydrogen ion.</p>

<p>What you will have will be this:</p>

<p align="center"><img src="asiteph2.gif" width="243" height="184"></p>
<pre>

</pre>
<p>You no longer have the ability to form ionic bonds between the substrate and the enzyme.  If those bonds were necessary to attach the  substrate and activate it in some way, then at this lower pH, the enzyme won't work.</p>
<pre>

</pre>
<p>What if you have a pH higher than 7 - in other words under alkaline conditions.</p>

<p>This time, the -COO<sup>-</sup> group won't be affected, but the -NH<sub>3</sub><sup>+</sup> group will lose a hydrogen ion.</p>

<p>That leaves . . .</p>

<p align="center"><img src="asiteph3.gif" width="243" height="185"></p>
<pre>

</pre>
<p>Again, there is no possibility of forming ionic bonds, and so the enzyme probably won't work this time either.</p>
</font>
</font></td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you think about it, this argument only works using ionic bonding if the enzyme has an optimum pH around 7.  Under fairly acidic or alkaline conditions, all the charges present in the enzyme and substrate would be the same - as in the examples above.</p>

<p>Under those circumstances, you would be looking at other sorts of bonding - hydrogen bonding, for example, perhaps involving hydrogen bonds between an ion on one of either the enzyme and substrate, and a suitable hydrogen atom or lone pair on the other.</p>

<p>The example given is easy to follow.  Don't worry about complications at this level.  As long as you can see why changing the pH might affect a simple case, that's all you need.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p>At extreme pH's, something more drastic can happen.  Remember that the tertiary structure of the protein is in part held together by ionic bonds just like those we've looked at between the enzyme and its substrate.</p>

<p>At very high or very low pH's, these bonds within the enzyme can be disrupted, and it can lose its shape.  If it loses its shape, the active site will probably be lost completely.  This is essentially the same as denaturing the protein by heating it too much.</p>
<pre>

</pre>
<p>This enzyme topic continues onto a final page about enzyme inhibitors.</p>
</font>
<pre>

</pre>
<table align="center" width="100%" cellpadding="5" border="1">
<tr><td bgcolor="#ccffcc"><font color="#000000" face="Helvetica, Arial" size="2">
<p><b>Questions to test your understanding</b></p>

<p>If this is the first set of questions you have done, please read the <a href="../../questions.html">introductory page</a> before you start.  You will need to use the BACK BUTTON on your browser to come back here afterwards.</p>

<p><a href="../questions/q-proteinenz2.pdf">questions on changing the conditions in enzyme catalysis</a></p>
<p><a href="../questions/a-proteinenz2.pdf">answers</a></p>

</font>
</td></tr>
</table>
<pre>

</pre>
<dl>
<font color="#006600" face="Arial, Helvetica " size="3">
<dt><p><b>Where would you like to go now?</b></p></dt>

<dd><p><a href="enzymes3.html#top"><b>To continue to the final page about enzymes . . .</b></a></p>
<p><a href="../aminoacidmenu.html#top"><b>To the amino acid and proteins menu . . .</b></a></p>
<p><a href="../../orgpropsmenu.html#top"><b>To the menu of other organic compounds . . .</b></a></p>
<p><a href="../../index.html#top"><b>To Main Menu . . .</b></a></p></dd>
</font></dl>

<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2007 (modified May 2016)</font></p>
</td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacids/enzymes2/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacids/enzymes/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>proteins as enzymes</title>

<meta name="description" content="An introduction to how some proteins function as enzymes, and the general features of enzyme catalysis">
<meta name="keywords" content="protein, proteins, structure, structures, enzyme, enzymes, catalyst, catalysis, active site, cofactor, coenzyme, inhibition, inhibitor, prosthetic group">

</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p><b>PROTEINS AS ENZYMES</b></p></font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="3">
<p>This page is an introduction to how proteins can work as enzymes - biological catalysts.  You should realise that this is written to cover the needs of a number of UK-based <i>chemistry</i> syllabuses for 16 - 18 year olds.  If you want detailed knowledge about enzymes for a biology or biochemistry course, you are probably in the wrong place!  This is just an introduction.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>This page follows on from a page about <a href="proteinstruct.html#top">protein structure</a>.  If you don't have a reasonable knowledge of the structure of proteins and the sorts of attractions that can be found in them, you may not understand bits of the present page.  Read the protein structure page first and come back here later.</p>
</font>
<font color="#ff0000" face="Helvetica, Arial" size="2">
<p><b>Important: &nbsp;</b>The specific examples of the enzymes that you will find on this page are only intended to give you a feel for the way that enzymes work.  Unless your syllabus specifically asks for a particular enzyme, there is no need for you to remember the details.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>Enzymes as catalysts</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p>Enzymes are mainly globular proteins - protein molecules where the tertiary structure has given the molecule a generally rounded, ball shape (although perhaps a very squashed ball in some cases).  The other type of proteins (fibrous proteins) have long thin structures and are found in tissues like muscle and hair.  We aren't interested in those in this topic.</p>

<p>These globular proteins can be amazingly active catalysts.  You are probably familiar with the use of catalysts like manganese(IV) oxide in decomposing hydrogen peroxide to give oxygen and water.  The enzyme catalase will also do this - but at a spectacular rate compared with inorganic catalysts.</p>

<p>One molecule of catalase can decompose almost a hundred thousand molecules of hydrogen peroxide every second.  That's very impressive!</p>

<p>This is a model of catalase, showing the globular structure - a bit like a tangled mass of string:</p>

<p align="center"><img src="catalase.jpg" width="400" height="338"></p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>This diagram was obtained from the <a href="http://www.rcsb.org/pdb/" target="_blank" rel="noopener">RCSB Protein Data Bank</a>.</p>

<p>You should be able to identify the alpha-helices and beta-pleated sheets.  If you can't, you obviously didn't read the page about <a href="proteinstruct.html#top">protein structure</a> mentioned above!</p>

<p>If you look very carefully, you might also spot two pink, non-protein structures hidden in the main structure.  More about these in a while . . .<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">

<p>An important point about enzymes is that they are very specific about what they can catalyse.  Even small changes in the reactant molecule can stop the enzyme from catalysing its reaction.  The reason for this lies in the <b><i>active site</i></b> present in the enzyme . . .</p>
<pre>

</pre>
<p><b>Active sites</b></p>

<p>Active sites are cracks or hollows on the surface of the enzyme caused by the way the protein folds itself up into its tertiary structure.  Molecules of just the right shape, and with just the right arrangement of attractive groups (see later) can fit into these active sites.  Other molecules won't fit or won't have the right groups to bind to the surface of the active site.</p>

<p>The usual analogy for this is a key fitting into a lock.  For the key to work properly it has to fit exactly into the lock.</p>

<p align="center"><img src="activesite1.gif" width="336" height="169"></p>

<p>In chemistry, we would describe the molecule which is actually going to react (the purple one in the diagram) as the <b><i>reactant</i></b>.  In biology and biochemistry, the reactant in an enzyme reaction is known instead as the <b><i>substrate</i></b>.
<pre>

</pre>
</p><p>You mustn't take this picture of the way a substrate fits into its enzyme too literally.  What is just as important as the physical shape of the substrate are the bonds which it can form with the enzyme.</p><p>

</p><p>Enzymes are protein molecules - long chains of amino acid residues.  Remember that sticking out all along those chains are the side groups of the amino acids - the &quot;R&quot; groups that we talked about on the page about protein structure.</p>

<p>Active sites, of course, have these &quot;R&quot; groups lining them as well - typically from about 3 to 12 in an active site.  The next diagram shows an imaginary active site:</p>

<p align="center"><img src="activesite2.gif" width="243" height="127"></p>
<pre>

</pre>
<p>Remember that these &quot;R&quot; groups contain the sort of features which are responsible for the tertiary structure in proteins.  For example, they may contain ionic groups like -NH<sub>3</sub><sup>+</sup> or -COO<sup>-</sup>, or -OH groups which can hydrogen bond, or hydrocarbon chains or rings which can contribute to van der Waals forces.</p>

<p>Groups like these help a substrate to attach to the active site - but only if the substrate molecule has an arrangement of groups in the right places to interact with those on the enzyme.</p>

<p>The diagram shows a possible set of interactions involving two ionic bonds and a hydrogen bond.</p>
<pre>

</pre>
<p align="center"><img src="activesite3.gif" width="243" height="147"></p>
<pre>

</pre>
<p>The groups shown with + or - signs are obvious.  The ones with the &quot;H&quot;s in them are groups capable of hydrogen bonding.  It is possible that one or more of the unused &quot;R&quot; groups in the active site could also be helping with van der Waals attractions between them and the substrate.</p>

<p>If the arrangement of the groups on the active site or the substrate was even slightly different, the bonding almost certainly wouldn't be as good - and in that sense, a different substrate wouldn't fit the active site on the enzyme.</p>

<p>This process of the catalyst reacting with the substrate and eventually forming products is often summarised as:</p>
<pre>

</pre>
<p align="center"><img src="summaryeqn.gif" width="405" height="15"></p>
<pre>

</pre>
<p>. . . where E is the enzyme, S the substrate and P the products.</p>

<p>The formation of the complex is reversible - the substrate could obviously just break away again before it converted into products.  The second stage is shown as one-way, but might be reversible in some cases.  It would depend on the energetics of the reaction.</p>
<pre>

</pre>
<p>So why does attaching itself to an enzyme increase the rate at which the substrate converts into products?</p>

<p>It isn't at all obvious why that should be - and most sources providing information at this introductory level just gloss over it or talk about it in vague general terms (which is what I am going to be forced to do, because I can't find a simple example to talk about!).</p>

<p>Catalysts in general (and enzymes are no exception) work by providing the reaction with a route with a lower activation energy.  Attaching the substrate to the active site must allow electron movements which end up in bonds breaking much more easily than if the enzyme wasn't there.</p>

<p>Strangely, it is much easier to see what might be happening in other cases where the situation is a bit more complicated . . .</p>
<pre>

</pre>
<p><b>Enzyme cofactors</b></p>

<p>What we have said so far is a major over-simplification for most enzymes.  Most enzymes aren't in fact just pure protein molecules.  Other non-protein bits and pieces are needed to make them work.  These are known as <b><i>cofactors</i></b>.</p>

<p>In the absence of the right cofactor, the enzyme doesn't work.  For those of you who like collecting obscure words, the inactive protein molecule is known as an <b><i>apoenzyme</i></b>.  When the cofactor is in place so that it becomes an active enzyme, it is called a <b><i>holoenzyme</i></b>.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you don't collect obscure words, don't worry about these!  Neither of them occurs in the syllabuses I am trying to cover with this material.  I have spent a lifetime in chemistry education without having come across either of them until researching this!<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p>There are two basically different sorts of cofactors.  Some are bound tightly to the protein molecule so that they become a part of the enzyme - these are called <b><i>prosthetic groups</i></b>.</p>

<p>Some are entirely free of the enzyme and attach themselves to the active site alongside the substrate - these are called <b><i>coenzymes</i></b>.</p>
<pre>

</pre>
<p><b><i>Prosthetic groups</i></b></p>

<p>Prosthetic groups can be as simple as a single metal ion bound into the enzyme's structure, or may be a more complicated organic molecule  (which might also contain a metal ion).  The enzymes <b><i>carbonic anhydrase</i></b> and <b><i>catalase</i></b> are simple examples of the two types.</p>
<pre>

</pre>
<p><i>Zinc ions in carbonic anhydrase</i></p>

<p>Carbonic anhydrase is an enzyme which catalyses the conversion of carbon dioxide into hydrogencarbonate ions (or the reverse) in the cell.  (If you look this up elsewhere, you will find that biochemists tend to persist in calling hydrogencarbonate by its old name, bicarbonate!)</p>

<p align="center"><img src="co2eqn.gif" width="263" height="18"></p>

<p>In fact, there are a whole family of carbonic anhydrases all based around different proteins, but all of them have a zinc ion bound up in the active site.  In this case, the mechanism is well understood and simple.  We'll look at this in some detail, because it is a good illustration of how enzymes work.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#ff0000" face="Helvetica, Arial" size="2">
<p><hr><b>Important: &nbsp;</b>This is just an example!  If you are doing a UK-based chemistry syllabus for 16 - 18 year olds, there is almost certainly no need to learn this.  If in doubt, check your syllabus.  If it doesn't explicitly ask for this reaction in detail, you don't  need to learn it.  To repeat - I'm just using it to illustrate how enzymes carry out a simple reaction.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p>The zinc ion is bound to the protein chain via three links to separate histidine residues in the chain - shown in pink in the picture of one version of carbonic anhydrase.  The zinc is also attached to an -OH group - shown in the picture using red for the oxygen and white for the hydrogen.</p>

<p align="center"><img src="caactivesite.jpg" width="200" height="190"></p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>This diagram comes from <a href="http://en.wikipedia.org/wiki/Carbonic_anhydrase" target="_blank" rel="noopener">Wikipedia</a>.  I have no reason to doubt its accuracy, but I can't guarantee it.</p>

<p>As far as I have been able to find out, all the various forms of carbonic anhydrase have the zinc ion bound to three histidine residues in this way - irrespective of what is happening in the rest of the protein molecule.  If I am wrong about this generalisation, could you please let me know via the address on the <a href="../../about.html">about this site</a> page.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p>The structure of the amino acid histidine is . . .</p>

<p align="center"><img src="histidine1.gif" width="90" height="145"></p>

<p>. . . and when it is a part of a protein chain, it is joined up like this:</p>

<p align="center"><img src="histidine2.gif" width="160" height="160"></p>

<p>If you look at the model of the arrangement around the zinc ion in the picture above, you should at least be able to pick out the ring part of the three molecules.</p>

<p>The zinc ion is bound to these histidine rings via dative covalent (co-ordinate covalent) bonds from lone pairs on the nitrogen atoms.  Simplifying the structure around the zinc . . .</p>

<p align="center"><img src="histidine3.gif" width="230" height="80"></p>

<p>The arrangement of the four groups around the zinc is approximately tetrahedral.  Notice that I have distorted the usual roughly tetrahedral arrangement of electron pairs around the oxygen - that's just to keep the diagram as clear as possible.</p>

<p>So that's the structure around the zinc.  How does this catalyse the reaction between carbon dioxide and water?</p>

<p>A carbon dioxide molecule is held by a nearby part of the active site so that one of the lone pairs on the oxygen is pointing straight at the carbon atom in the middle of the carbon dioxide molecule.  Attaching it to the enzyme also increases the existing polarity of the carbon-oxygen bonds.</p>

<p align="center"><img src="camech1.gif" width="349" height="81"></p>

<p>If you have done any work on organic reaction mechanisms at all, then it is pretty obvious what is going to happen.  The lone pair forms a bond with the carbon atom and part of one of the carbon-oxygen bonds breaks and leaves the oxygen atom with a negative charge on it.</p>

<p align="center"><img src="camech2.gif" width="151" height="81"></p>

<p>What you now have is a hydrogencarbonate ion attached to the zinc.</p>

<p>The next diagram shows this broken away and replaced with a water molecule from the cell solution.</p>

<p align="center"><img src="camech3.gif" width="275" height="154"></p>

<p>All that now needs to happen to get the catalyst back to where it started is for the water to lose a hydrogen ion.  This is transferred by another water molecule to a nearby amino acid residue with a nitrogen in the &quot;R&quot; group - and eventually, by a series of similar transfers, out of the active site completely.</p>

<p>. . . and the carbonic anhydrase enzyme can do this sequence of reactions about a million times a second.  This is a wonderful piece of molecular machinery!</p>

<p>Let me repeat yet again:  If you are doing a UK-based chemistry exam  for 16 - 18 year olds, you are unlikely to need details of this reaction.  I've talked it through in some detail to show that although enzymes are complicated molecules, all they do is some basic chemistry.  It is just that this particular example is a lot easier to understand than most!</p>
<pre>

</pre>
<p><i>The haem (US: heme) group in catalase</i></p>

<p>Remember the model of catalase from further up the page . . .</p>

<p align="center"><img src="catalase.jpg" width="400" height="338"></p>

<p>At the time, I mentioned the non-protein groups which this contains, shown in pink in the picture.  These are <b><i>haem</i></b> (US: heme) groups bound to the protein molecule, and an essential part of the working of the catalase.  The haem group is a good example of a prosthetic group.  If it wasn't there, the protein molecule wouldn't work as a catalyst.</p>

<p>The haem groups contain an iron(III) ion bound into a ring molecule - one of a number of related molecules called porphyrins.  The iron is locked into the centre of the porphyrin molecule via dative covalent bonds from four nitrogen atoms in the ring structure.</p>

<p>There are various types of porphyrin, so there are various different haem groups.  The one we are interested in is called haem B, and a model of the haem B group (with the iron(III) ion in grey at the centre) looks like this:</p>

<p align="center"><img src="haemb.jpg" width="200" height="207"></p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>This diagram comes from <a href="http://en.wikipedia.org/wiki/Heme" target="_blank" rel="noopener">Wikipedia</a>, and you will also find a proper structure for the group on the page you will get to by following this link if you are interested (and a lot more information that you probably won't want to know about!).</p>

<p>You may have come across haemoglobin in the transport of oxygen around the blood.  This is the same haem group that is at the heart of that - with one small difference.  In haemoglobin, the iron is present as iron(II) rather than iron(III).<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p>The reaction that catalase carries out is the decomposition of hydrogen peroxide into water and oxygen.</p>

<p align="center"><img src="h2o2eqn1.gif" width="212" height="15"></p>

<p>A lot of work has been done on the mechanism for this reaction, but I am only going to give you a simplified version rather than describe it in full.  Although it looks fairly simple on the surface, there are a lot of hidden things going on to complicate it.</p>

<p>Essentially the reaction happens in two stages and involves the iron changing its oxidation state.  An easy change of oxidation state is one of the main characteristics of transition metals.  In the lab, iron commonly has two oxidation states (as well as zero in the metal itself), +2 and +3, and changes readily from one to the other.</p>

<p>In catalase, the change is from +3 to the far less common +4 and back again.</p>

<p>In the first stage there is a reaction between a hydrogen peroxide molecule and the active site to give:</p>

<p align="center"><img src="h2o2eqn2.gif" width="365" height="15"></p>

<p>The &quot;Enzyme&quot; in the equation refers to everything (haem group and protein) apart from the iron ion.  The &quot;(III)&quot; and &quot;(IV)&quot; are the oxidation states of the iron in both cases.  This equation (and the next one) are NOT proper chemical equations.  They are just summaries of the most obvious things which have happened.</p>

<p>The new arrangement around the iron then reacts with a second hydrogen peroxide to regenerate the original structure and produce oxygen and a second molecule of water.</p>

<p align="center"><img src="h2o2eqn3.gif" width="397" height="15"></p>

<p>What is hidden away in this simplification are the other things that are happening at the same time - for example, the rest of the haem group and some of the amino acid residues around the active site are also changed during each stage of the reaction.</p>

<p>And if you think about what has to happen to the hydrogen peroxide molecule in both reactions, it has to be more complicated than this suggests.  Hydrogen peroxide is joined up as H-O-O-H, and yet both hydrogens end up attached to the same oxygen.  That is quite a complicated thing to arrange in small steps in a mechanism, and involves hydrogen ions being transferred via amino acids residues in the active site.</p>

<p>So do you need to remember all this for chemistry purposes at this level?  No - not unless your syllabus specifically asks you for it.  It is basically just an illustration of the term &quot;prosthetic group&quot;.</p>

<p>It also shows that even in a biochemical situation, transition metals behave in the same sort of way as they do in inorganic chemistry - they form complexes, and they change their oxidation state.</p>

<p>And if you want to follow this up to look in detail at what is happening, you will find the same sort of interactions around the active site that we looked at in the simpler case of carbonic anydrase.  (But please don't waste time on this unless you have to - it is seriously complicated!)</p>
<pre>

</pre>
<p><b><i>Coenzymes</i></b></p>

<p>Coenzymes are another form of cofactor.  They are different from prosthetic groups in that they aren't permanently attached to the protein molecule.  Instead, coenzymes attach themselves to the active site alongside the substrate, and the reaction involves both of them.  Once they have reacted, they both leave the active site - both changed in some way.</p>

<p>A simple diagram showing a substrate and coenzyme together in the active site might look like this:</p>

<p align="center"><img src="activesite4.gif" width="149" height="179"></p>

<p>It is much easier to understand this with a (relatively) simple example.</p>
<pre>

</pre>
<p><i>NAD+ as coenzyme with alcohol dehydrogenase</i></p>

<p>Alcohol dehydrogenase is an enzyme which starts the process by which alcohol (ethanol) in the blood is oxidised to harmless products.  The name &quot;dehydrogenase&quot; suggests that it is oxidising the ethanol by removing hydrogens from it.</p>

<p>The reaction is actually between ethanol and the coenzyme NAD+ attached side-by-side to the active site of the protein molecule.  NAD+ is a commonly used coenzyme in all sorts of redox reactions in the  cell.</p>

<p>NAD+ stands for <b><i>nicotinamide adenine dinucleotide</i></b>.  The plus sign which is a part of its name is because it carries a positive charge on a nitrogen atom in the structure.</p>

<p>The &quot;nicotinamide&quot; part of the structure comes from the vitamin variously called vitamin B3, niacin or nicotinic acid.  Several important coenzymes are derived from vitamins.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>I'm not going to confuse you with the structures of NAD+ or even nicotinic acid - this page is already long enough!  If you are interested, they are easy to find via a Google search.  In common with what I have done on the rest of this page, this is just an example to illustrate how a coenzyme works which is reasonably easy to understand.  You are unlikely to need details for any chemistry exam at this level.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p>Ethanol is oxidised by a reaction with NAD+ helped by the active site of the enzyme.  At  the end of the reaction, ethanal (acetaldehyde) is formed, and the NAD+ has been converted into another compound known as NADH.</p>

<p align="center"><img src="ethanolnad.gif" width="379" height="50"></p>

<p>As far as the NAD+ is concerned, it has picked up a hydrogen atom together with an extra electron which has neutralised the charge.</p>

<p>Both major products - ethanal and NADH - leave the active site and are processed further in other cell reactions.</p>

<p>The very poisonous ethanal is oxidised at once to ethanoic acid using a different enzyme, but again using NAD+ as the coenzyme.  And the ethanoic acid from that reacts on through a whole set of further enzyme-controlled reactions to eventually end up as carbon dioxide and water.</p>

<p>What about the NADH?  This is a coenzyme in its own right, and takes part in reactions where something needs reducing.  The hydrogen atom and the extra electron that it picked up from the ethanol are given to something else.  In the process, of course, the NADH gets oxidised back to NAD+ again.</p>

<p>In general terms, for a substrate S which needs reducing:</p>

<p align="center"><img src="nadheqn.gif" width="341" height="15"></p>

<p>And one final time - do you need to remember any of this?  No, unless this particular example is on your syllabus.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>Because this page is getting so long, and because there is still quite a bit of enzyme chemistry to talk about, it continues on another two pages.  You will find the link to the first of these below.</p>

<p>What follows is a page about the effect of substrate concentration, temperature and pH on enzymes, and then a further page about enzyme inhibitors.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<pre>

</pre>
<table align="center" width="100%" cellpadding="5" border="1">
<tr><td bgcolor="#ccffcc"><font color="#000000" face="Helvetica, Arial" size="2">
<p><b>Questions to test your understanding</b></p>

<p>If this is the first set of questions you have done, please read the <a href="../../questions.html">introductory page</a> before you start.  You will need to use the BACK BUTTON on your browser to come back here afterwards.</p>

<p><a href="../questions/q-proteinenz1.pdf">questions on proteins as enzymes</a></p>
<p><a href="../questions/a-proteinenz1.pdf">answers</a></p>

</font>
</td></tr>
</table>
<pre>

</pre>
<dl>
<font color="#006600" face="Arial, Helvetica " size="3">
<dt><p><b>Where would you like to go now?</b></p></dt>

<dd><p><a href="enzymes2.html#top"><b>To continue to the next page about enzymes . . .</b></a></p>
<p><a href="../aminoacidmenu.html#top"><b>To the amino acid and proteins menu . . .</b></a></p>
<p><a href="../../orgpropsmenu.html#top"><b>To the menu of other organic compounds . . .</b></a></p>
<p><a href="../../index.html#top"><b>To Main Menu . . .</b></a></p></dd>
</font></dl>

<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2007 (modified May 2016)</font></p>
</td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacids/enzymes/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacids/dna6/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>DNA - mutations</title>

<meta name="description" content="A  very brief look at how the code in DNA can go wrong and the effect that that would have on the proteins it codes for.">
<meta name="keywords" content="DNA, code, base, sequence, codon, amino acid, mutation">

</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p><b>DNA - MUTATIONS</b></p></font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="3">
<p>This page takes a very brief look at what happens if the code in DNA becomes changed in some way, and the effect that would have on the proteins it codes for.  It is designed for 16 - 18 year old <i>chemistry</i> students.  In fact, most chemistry students won't need this - check your syllabus and past papers before you go on.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you have come straight to this page from a search engine, you should be aware that this is the final page in a sequence of pages about DNA, RNA and protein synthesis, starting with the <a href="dna1.html#top">structure of DNA</a>.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>Random changes to the genetic code</b></p>
</font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p>Copying errors when DNA replicates or is transcribed into RNA can cause changes in the sequence of bases which makes up the genetic code.  Radiation and some chemicals can also cause changes.  The examples which follow show some of the easier-to-understand effects of this.</p>
<pre>

</pre>
<p><b>Changes to individual bases</b></p>

<p>Remember that a set of three bases in a gene in DNA codes for a particular amino acid.  If you have followed this sequence of pages from the beginning, you will have come across this table showing the codons in DNA:</p>
<pre>

</pre>
<p align="center"><img src="dnacode.gif" width="453" height="397"></p>
<pre>

</pre>
<p>A gene will be made up of a string of these codes rather like a string of 3-letter words in a sentence.  We'll use that as a simple analogy.  Take the sentence:</p>
</font>

<font color="#ff0000" face="Helvetica, Arial" size="3">
<p align="center">the big fox bit the dog but not the boy</p>
</font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p>Suppose one letter got changed in this by accident.  Suppose, for example, the &quot;d&quot; in dog got replaced by a &quot;p&quot;.  The sentence would now read:</p>
</font>

<font color="#ff0000" face="Helvetica, Arial" size="3">
<p align="center">the big fox bit the pog but not the boy</p>
</font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p>Clearly this doesn't make complete sense any more.  Would that matter if the same thing happened in a gene?  It depends!</p>

<p>If you look back at the table, there are several amino acids which are coded for by more than one base combination.  For example, glycine (Gly) is coded for by GGT, GGC, GGA and GGG.  It doesn't matter what the last base is - you would get glycine whatever base followed the initial GG.</p>

<p>That means that a mutation at the end of a codon like this wouldn't make any difference to the protein chain which would eventually form.  These are known as <b><i>silent mutations</i></b>.</p>

<p>Alternatively, of course, you could well get a code for a different amino acid or even a stop codon.</p>

<p>If a stop codon was produced in the middle of the gene, then the protein formed would be too short, and almost certainly wouldn't function properly.</p>

<p>If a different amino acid was produced, how much it mattered would depend on whereabouts it was in the protein chain.  If it was near the active site of an enzyme, for example, it might stop the enzyme from working entirely.</p>

<p>On the other hand, if it was on the outside of an enzyme, and didn't affect the way the  protein chain folded, it might not matter at all.</p>
<pre>

</pre>
<p><b>Inserting or deleting bases</b></p>

<p>The situation is more dramatic if extra bases are inserted into the code, or some bases are deleted from the code.  Using our example sentence from above, and keeping the three letter word structure:</p>

<p>If you insert a single extra base:</p>
</font>

<font color="#ff0000" face="Helvetica, Arial" size="3">
<p align="center">the big fro xbi tth edo gbu tno tth ebo y</p>
</font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p>An extra &quot;r&quot; is inserted in &quot;fox&quot;.  If the sentence still has to be read three letters at a time (as in DNA), everything from then on becomes completely meaningless.</p>

<p>If you delete a single base:</p>
</font>

<font color="#ff0000" face="Helvetica, Arial" size="3">
<p align="center">the big fxb itt hed ogb utn ott heb oy</p>
</font>

<font color="#000000" face="Helvetica, Arial" size="3">

<p>This time the &quot;o&quot; in &quot;fox&quot; has been deleted.  And again, because we have to read the  letters in groups of three, the rest of the sentence becomes completely wrecked.</p>

<p>So does this matter?  Well, of course it does!  Large chunks of the protein will consist of completely wrong amino acid residues.</p>

<p>We've looked so far at inserting or deleting <i>one</i> base.  What if you do it for more than one?</p>

<p>The effect is the same unless you add or delete multiples of three bases - without changing any other codons.    If you added an extra three bases <i>between two existing codons</i>, then essentially you are just adding an extra word.</p>
</font>

<font color="#ff0000" face="Helvetica, Arial" size="3">
<p align="center">the big fox bit the xjy dog but not the boy</p>
</font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p>That extra word represents an extra codon in the DNA, and so an extra amino acid residue in the protein chain.  Does this matter?  It depends where it is in the chain (Is it important to the active site of an enzyme, for example?), and whether it affects the folding of the chain.</p>

<p>What if the three bases were inserted so that they broke up an existing codon?  Here is the same extra &quot;word&quot;, &quot;xjy&quot;, dropped in the word &quot;bit&quot;.  Everything is then reshuffled into groups of three letters.</p>
</font>

<font color="#ff0000" face="Helvetica, Arial" size="3">
<p align="center">the big fox bxj yit the dog but not the boy</p>
</font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p>You can see that the effect is again fairly limited.  It will change one codon completely, and introduce an extra codon.  That would give you one different amino acid and one extra amino acid in the chain.  Again, how much that would affect the final protein depends on where it happens in the chain.</p>

<p>Deleting a whole codon again leaves most of the protein chain unchanged.  Again, whether the function of the protein is affected depends on where the  missing amino acid should have been and how critical it was to the way the protein folded.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Important: &nbsp;</b>The material on remainder of this page was written for a specific UK syllabus which no longer requires it.  I have left it here in case it is useful to anyone else.  Read it out of interest if you want to, but check whatever syllabus you are doing before you waste any time learning it.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>Some diseases caused by mutation</b></p>
</font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p>The following examples illustrate some of the changes we've looked at above and how they can result in disease.</p>
<pre>

</pre>
<p><b>Cystic fibrosis</b></p>

<p>Cystic fibrosis is an inherited disease which affects the lungs and digestive system.  It results from mutation in a gene responsible for making a protein which is involved in the transport of ions across cell boundaries.</p>

<p>The effect is to produce a sticky mucus which clogs the lungs and can lead to serious infection.  A similar sticky mucus also blocks the pancreas (a part of the digestive system) which provides enzymes for breaking down food.  This gets in the way of the processes which convert the food into molecules which can be absorbed by the body.</p>

<p>There are lots of different mutations which can cause this, but we'll just have a quick look at the one which accounts for about 70% of cystic fibrosis cases.</p>

<p>The base sequence in the part of the gene affected ought to look like this:</p>

<p align="center"><img src="cf1.gif" width="268" height="42"></p>

<p>The phenylalanine (Phe) in red is the amino acid which is missing from the final protein in many sufferers from cystic fibrosis.  However, it isn't quite as simple as just losing the TTT codon.</p>

<p>Instead, the three bases lost are:</p>

<p align="center"><img src="cf2.gif" width="268" height="92"></p>

<p>That leaves the sequence:</p>

<p align="center"><img src="cf3.gif" width="231" height="42"></p>

<p>Notice that the amino acid sequence is identical to before but without the phenylalanine.  How did that happen when we didn't actually remove the whole of the phenylalanine codon?</p>

<p>If you look carefully, you will see that the codon for the second isoleucine (Ile) is different from before.  It so happens that isoleucine is coded for by both ATC and ATT.  Once the second T (the red one) has joined the existing AT, all the rest of the base sequence is exactly what it was before.</p>

<p><b>Sickle cell anaemia (US: anemia)</b></p>

<p>This is so called because red blood cells change their shape from the normal flexible doughnut shape to a much more rigid sickle shape - rather like a crescent moon.</p>

<p>It results from the change of a single base in a gene responsible for making one of the protein chains which makes up haemoglobin (US: hemoglobin).</p>

<p>The affected part of the gene should read:</p>

<p align="center"><img src="sca1.gif" width="323" height="42"></p>

<p>What it actually reads in someone suffering from sickle cell anaemia is:</p>

<p align="center"><img src="sca2.gif" width="321" height="42"></p>
<p>The effect of this single change is to make the haemoglobin temporarily polymerise to make fibres after it has released the oxygen that it carries around the body.  This changes the shape of the red blood cells so that they don't flow so easily - it makes them sticky, especially in small blood vessels.  This can cause pain and lead to organ damage.</p>
<pre>

</pre>
<p><b>Haemophilia (US: hemophilia)</b></p>

<p>Sufferers from haemophilia lack a protein in the blood which allows it to clot.  That means that if someone with haemophilia cuts themselves, the wound will just continue to bleed.</p>

<p>There are all sorts of mutations which cause haemophilia.  One which is easy to understand is caused by changing a single base at the beginning of a codon for argenine (CGA) somewhere in the gene to give TGA.  If you look back to the table higher up the page, you will find that TGA is a stop codon.</p>

<p>All that will be produced is a useless fragment of the intended protein.</p>
</font>
<pre>

</pre>
<table align="center" width="100%" cellpadding="5" border="1">
<tr><td bgcolor="#ccffcc"><font color="#000000" face="Helvetica, Arial" size="2">
<p><b>Questions to test your understanding</b></p>

<p>If this is the first set of questions you have done, please read the <a href="../../questions.html">introductory page</a> before you start.  You will need to use the BACK BUTTON on your browser to come back here afterwards.</p>

<p><a href="../questions/q-dna6.pdf">questions on DNA mutations</a></p>
<p><a href="../questions/a-dna6.pdf">answers</a></p>

</font>
</td></tr>
</table>
<pre>

</pre>
<dl>
<font color="#006600" face="Arial, Helvetica " size="3">
<dt><p><b>Where would you like to go now?</b></p></dt>

<dd>
<p><a href="../aminoacidmenu.html#top"><b>To the amino acid and other biochemistry menu . . .</b></a></p>
<p><a href="../../orgpropsmenu.html#top"><b>To the menu of other organic compounds . . .</b></a></p>
<p><a href="../../index.html#top"><b>To Main Menu . . .</b></a></p></dd>
</font></dl>

<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2007 (modified May 2016)</font></p>
</td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacids/dna6/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacids/dna5/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>protein synthesis - from mRNA to protein</title>

<meta name="description" content="How the code in messenger RNA gets converted into an actual protein molecule.">
<meta name="keywords" content="messenger, transfer, rna, structure, genetic, code, base, sequence, codon, anti-codon, translation, protein, ribosome, amino acid">

</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p><b>PROTEIN SYNTHESIS</b></p></font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="3">
<p>This page looks at how the information coded in messenger RNA is used to build protein chains.  It is designed for 16 - 18 year old <i>chemistry</i> students.  If you are a biochemistry or biology student, you will probably find it a useful introduction, but will have to look elsewhere to find all the detail you need.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you have come straight to this page from a search engine, you should be aware that this is the fifth page in a sequence of pages about DNA and RNA.  It will all make more sense if you start from the beginning of the sequence with the <a href="dna1.html#top">structure of DNA</a>.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="4">

<p><b>From messenger RNA to a protein chain</b></p>
</font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p><b>A quick overview of the process</b></p>

<p>You will remember that messenger RNA contains a sequence of bases which, read three at a time, code for the amino acids used to make protein chains.  Each of the sets of three bases is known as a <b><i>codon</i></b>.  The table below repeats one from the previous page:</p>
<pre>

</pre>
<p align="center"><img src="rnacode.gif" width="453" height="397"></p>
<pre>

</pre>
<p>You may also remember that three codons serve as <b><i>stop codons</i></b>, and one (AUG) codes for methionine, but also serves as a <b><i>start codon</i></b>.  We'll look in some detail at how this works further down the page.</p>

<p>Translating the code into an actual protein chain is complicated by the fact that individual amino acids won't interact with the messenger RNA chain.  The amino acids have to be carried to the messenger RNA by another type of RNA known as transfer RNA - abbreviated to tRNA (as opposed to mRNA for messenger RNA).</p>

<p>All of this is controlled by a <b><i>ribosome</i></b> - a hugely complicated structure involving protein molecules and yet another form of RNA (ribosomal RNA or rRNA).</p>

<p>This is going to be quite complicated.  We'll take it gently and simplify it where possible.</p>
<pre>

</pre>
<p><b>Finding the start point of the messenger RNA</b></p>

<p>You may find descriptions of this process which imply (although without actually saying so directly) that the messenger RNA starts with the codon AUG - the start codon - at the 5' end.  That's not so!</p>

<p>There is a length of RNA upstream of the start codon which isn't actually used to build the protein chain.  So how does the system know where to start?  How does it find the right AUG codon from all the  ones which are probably strung out along the RNA to code for the amino acid methionine?</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you don't know what I mean by the <i>5' end</i> or by <i>upstream</i>, it is probably because you haven't read these pages from the beginning.  Taking short cuts is rarely a good idea!<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p>Ribosomes come in two parts - a smaller bit and a larger bit.  The smaller bit is involved in finding the start point.</p>

<p>It attaches to the 5' end of the messenger RNA and moves along it until it comes to a particular pattern of bases which it can bind to.  This pattern occurs just before the first occurrence of the AUG codon in the messenger RNA strand.</p>

<p>The ribosome now has to build the protein chain starting with a methionine at the AUG codon it has just found.</p>

<p>Before we can talk about that we have to introduce transfer RNA . . .</p>
<pre>

</pre>
<p><b>Transfer RNA</b></p>

<p>Transfer RNA (tRNA) is responsible for carrying amino acids to the messenger RNA and then holding them there in a way that enables them to join together.</p>

<p>Transfer RNA is a short bit of RNA containing about 80 or so bases.  These are mostly the same bases as in messenger RNA (A, U, G and C), but it also contains some modified bases which won't concern us at this level.  A model of a typical transfer RNA looks like this:</p>
<pre>

</pre>
<p align="center"><img src="trna1.gif" width="301" height="300"></p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>This diagram comes from <a href="http://en.wikipedia.org/wiki/Transfer_RNA" target="_blank" rel="noopener">Wikipedia</a>.  Most of the colour coding is irrelevant to this discussion - but note the little bit in grey at the bottom which is where the anti-codon is (see below).<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p>This model is quite difficult to follow, and I am going to simplify it down to pick out the two important bits of it</p>

<p>At the 3' end of every transfer RNA molecule, the chain ends with the sequence of bases C C A.  Remember that the bases in RNA and DNA are attached to a backbone of alternating phosphate and sugar groups.  At the very end of the chain is the -OH group on the 3' carbon of a ribose ring.</p>

<p>The amino acid gets attached to this by forming an ester link between this -OH group and the -COOH group of the amino acid.  This is no different from the formation of an ester between, say, ethanol and ethanoic acid - except that it is carried out under the influence of an enzyme rather than the more fierce conditions used in the lab.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>How the amino acid attaches to the tRNA is just for interest.  It is unlikely to be needed for exam purposes at this level.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p>The other important bit is at the bottom of the molecule, shown in grey in the model. This is known as an <b><i>anti-codon</i></b>.  As you will see shortly, the anti-codon attaches the transfer RNA with its amino acid to the right place on the messenger RNA molecule.</p>

<p>For chemistry purposes, all we are interested in is the attached amino acid, and the anti-codon, so we can simplify the whole thing down. Here is a very simplified diagram showing the transfer RNA for methionine with the methionine attached:</p>

<p align="center"><img src="trna3.gif" width="241" height="169"></p>
<pre>

</pre>
<p>In the diagram, the anti-codon is for the amino acid methionine.  The messenger RNA code for methionine is AUG.  If you look at the code in the anti-codon for methionine, it is UAC.  That is exactly complementary to AUG.  The U in the anti-codon will pair with the A in the messenger RNA; the A in the anti-codon pairs with the U in the mRNA; and the C in the anti-codon pairs with the G in the mRNA.</p>

<p>How does a transfer RNA molecule pick up the right amino acid?  This is all under control of enzymes which recognise the shapes of the various amino acid and tRNA molecules and make sure that they pair up properly.</p>
<pre>

</pre>
<p><b>Translation</b></p>

<p><b><i>Translation</i></b> is the name given to the process of turning the coded message in the messenger RNA into the final protein chain.</p>

<p>We left the messenger RNA a little while back with part of a ribosome attached to it at the AUG start codon.  The diagram shows this, together with a small part of the RNA base sequence downstream of the start codon needed to make an imaginary protein chain.  The bases upstream of the start codon aren't relevant to us once the ribosome has found the place to start from.</p>

<p>None of these diagrams are drawn to scale!</p>

<p align="center"><img src="translate1.gif" width="288" height="97"></p>

<p>Now two things happen.  The transfer RNA carrying a methionine attaches itself to the AUG codon by pairing its anti-codon bases with the complementary bases on the messenger RNA.  And the second, bigger part of the ribosome attaches to the system as well.</p>

<p align="center"><img src="translate2.gif" width="370" height="227"></p>

<p>Now another transfer RNA molecule with its attached amino acid binds to the next codon along the chain.  The next codon on the messenger RNA is GGU which codes for glycine (Gly).  The anti-codon would therefore have to be CCA.  Remember that A pairs with U, and G pairs with C.</p>

<p align="center"><img src="translate3.gif" width="335" height="227"></p>

<p>Next, the ribosome moves along the messenger RNA chain to the next codon.  At the same time a peptide bond is made between the two amino acids, and the first one (the methionine) breaks away from its transfer RNA.</p>

<p>That transfer RNA molecule leaves the ribosome and goes off to pick up another methionine.</p>

<p align="center"><img src="translate4.gif" width="293" height="227"></p>

<p>Now the process repeats.  The next codon is GUA which codes for valine (Val).  The anti-codon must be CAU.  (If you can't see this at once, stop and think about it.  Don't go on until you are happy that you could work out the anti-codon for every codon, and vice versa.)</p>

<p align="center"><img src="translate5.gif" width="293" height="227"></p>

<p>And again, the ribosome moves forward one codon, a new peptide bond is formed, and the  transfer RNA on the left breaks away to be used again later.</p>

<p align="center"><img src="translate5a.gif" width="329" height="227"></p>

<p>And the next transfer RNA with its amino acid comes along . . .</p>

<p align="center"><img src="translate6.gif" width="327" height="227"></p>

<p>. . . and so on, and on and on . . .</p>

<p>Eventually, of course, this will come to an end.  How?</p>

<p>Eventually, the ribosome will come to a stop codon.  The three stop codons don't code for any amino acids, and so the process comes to a halt.  The protein chain produced up to that point is then released from the ribosome, and then folds itself up into its secondary and tertiary structures.</p>
<pre>

</pre>
<p>The final page in this sequence looks very briefly at what happens when the code in DNA becomes changed in some way . . .</p>
</font>
<pre>

</pre>
<table align="center" width="100%" cellpadding="5" border="1">
<tr><td bgcolor="#ccffcc"><font color="#000000" face="Helvetica, Arial" size="2">
<p><b>Questions to test your understanding</b></p>

<p>If this is the first set of questions you have done, please read the <a href="../../questions.html">introductory page</a> before you start.  You will need to use the BACK BUTTON on your browser to come back here afterwards.</p>

<p><a href="../questions/q-dna5.pdf">questions on protein synthesis</a></p>
<p><a href="../questions/a-dna5.pdf">answers</a></p>

</font>
</td></tr>
</table>
<pre>

</pre>
<dl>
<font color="#006600" face="Arial, Helvetica " size="3">
<dt><p><b>Where would you like to go now?</b></p></dt>
<dd><p><a href="dna6.html#top"><b>To the final page about DNA . . .</b></a></p>
<p><a href="../aminoacidmenu.html#top"><b>To the amino acid and other biochemistry menu . . .</b></a></p>
<p><a href="../../orgpropsmenu.html#top"><b>To the menu of other organic compounds . . .</b></a></p>
<p><a href="../../index.html#top"><b>To Main Menu . . .</b></a></p></dd>
</font></dl>

<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2007 (last modified May 2016)</font></p>
</td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacids/dna5/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacids/dna4/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>the genetic code</title>

<meta name="description" content="How the base sequences in DNA and RNA code for particular amino acids.">
<meta name="keywords" content="rna, dna, structure, genetic, code, base, sequence, codon">

</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p><b>THE GENETIC CODE</b></p></font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="3">
<p>This page looks at how the base sequences in DNA and RNA are used to code for particular amino acids when it comes to building protein chains.  It is designed for 16 - 18 year old <i>chemistry</i> students.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you have come straight to this page from a search engine, you should be aware that this is the fourth page in a sequence of pages about DNA and RNA.  Unless you just want a quick reference to get the coding for a particular amino acid, it would pay you to start from the beginning with the <a href="dna1.html#top">structure of DNA</a>.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">

<p><b>Introduction</b></p>

<p>You can think of the sequences of bases in the coding strand of DNA or in messenger RNA as coded instructions for building protein chains out of amino acids.  There are 20 amino acids used in making proteins, but only four different bases to be used to code for them.</p>

<p>Obviously one base can't code for one amino acid.  That would leave 16 amino acids with no codes.</p>

<p>If you took two bases to code for each amino acid, that would still only give you 16 possible codes (TT, TC, TA, TG, CT, CC, CA and so on) - still not enough.</p>

<p>However, if you took three bases per amino acid, that gives you 64 codes (TTT, TTC, TTA, TTG, TCT, TCC and so on).  That's enough to code for everything with lots to spare.  You will find a full table of these below.</p>

<p>A three base sequence in DNA or RNA is known as a <b><i>codon</i></b>.</p>
<pre>

</pre>
<p><b>The code in DNA</b></p>

<p>The codes in the coding strand of DNA and in messenger RNA aren't, of course, identical, because in RNA the base uracil (U) is used instead of thymine (T).</p>

<p>The table shows how the various combinations of three bases  in the coding strand of DNA are used to code for individual amino acids - shown by their three letter abbreviation.</p>
<pre>

</pre>
<p align="center"><img src="dnacode.gif" width="453" height="397"></p>
<pre>

</pre>
<p>The table is arranged in such a way that it is easy to find any particular combination you want.  It is fairly obvious how it works and, in any case, it doesn't take very long just to scan through the table to find what you want.</p>

<p>The colours are to stress the fact that most of the amino acids have more than one code.  Look, for example, at leucine in the first column.  There are six different codons all of which will eventually produce a leucine (Leu) in the protein chain.  There are also six for serine (Ser).</p>

<p>In fact there are only two amino acids which have only one sequence of bases to code for them - methionine (Met) and tryptophan (Trp).</p>

<p>You have probably noticed that three codons don't have an amino acid written beside them, but say &quot;stop&quot; instead.  For obvious reasons these are known as <b><i>stop codons</i></b>.  We'll leave talking about those until we have looked at the way the code works in messenger RNA.</p>
<pre>

</pre>
<p><b>The code in messenger RNA</b></p>

<p>You will remember that when DNA is transcribed into messenger RNA, the sequence of bases remains exactly the same, except that each thymine (T) is replaced by uracil (U).  That gives you the table:</p>
<pre>

</pre>
<p align="center"><img src="rnacode.gif" width="453" height="397"></p>
<pre>

</pre>
<p>In many ways, this is the more useful table.  Messenger RNA is directly involved in the production of the protein chains (see the next page in this sequence).  The DNA coding chain is one stage removed from this because it must first be transcribed into a messenger RNA chain.</p>
<pre>

</pre>
<p><b>Start and stop codons</b></p>

<p>The stop codons in the RNA table (UAA, UAG and UGA) serve as a signal that the end of the chain has been reached during protein synthesis - and we will come back to that on the next page.</p>

<p>There is also a <b><i>start codon</i></b> - but you won't find it called that in the table!</p>

<p>The codon that marks the start of a protein chain is AUG.  If you check the table, that's the amino acid, methionine (Met).  That ought to mean that every protein chain must start with methionine.  That's not quite true because in some cases the methionine can get chopped off the chain after synthesis is complete.</p>
</font>
<pre>

</pre>
<table align="center" width="100%" cellpadding="5" border="1">
<tr><td bgcolor="#ccffcc"><font color="#000000" face="Helvetica, Arial" size="2">
<p><b>Questions to test your understanding</b></p>

<p>If this is the first set of questions you have done, please read the <a href="../../questions.html">introductory page</a> before you start.  You will need to use the BACK BUTTON on your browser to come back here afterwards.</p>

<p><a href="../questions/q-dna4.pdf">questions on DNA - the genetic code</a></p>
<p><a href="../questions/a-dna4.pdf">answers</a></p>

</font>
</td></tr>
</table>
<pre>

</pre>
<dl>
<font color="#006600" face="Arial, Helvetica " size="3">
<dt><p><b>Where would you like to go now?</b></p></dt>


<dd><p><a href="dna5.html#top"><b>To the next page about protein synthesis . . .</b></a></p>
<p><a href="../aminoacidmenu.html#top"><b>To the amino acid and other biochemistry menu . . .</b></a></p>
<p><a href="../../orgpropsmenu.html#top"><b>To the menu of other organic compounds . . .</b></a></p>
<p><a href="../../index.html#top"><b>To Main Menu . . .</b></a></p></dd>
</font></dl>

<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2007 (last modified May 2016)</font></p>
</td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacids/dna4/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacids/dna3/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>transcription - from DNA to RNA</title>

<meta name="description" content="A simple look at the structure of RNA and how the information in DNA is used to make messenger RNA">
<meta name="keywords" content="rna, dna, structure, deoxyribonucleic acid, ribonucleic acid, base, nucleotide, transcribe, transcription, uracil, adenine, thymine, guanine, cytosine, polymerase">

</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p><b>TRANSCRIPTION - FROM DNA TO RNA</b></p></font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="3">
<p>This page takes a simple look at the structure of RNA and how the information in DNA is used to make messenger RNA.  It is designed for 16 - 18 year old <i>chemistry</i> students, and if you are doing biology or biochemistry, you will probably need more detail than this page gives.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you have come straight to this page from a search engine), you should be aware that this is the third page in a sequence of pages about DNA.  These pages are written to be read one after the other, so unless you already understand the <a href="dna1.html#top">structure of DNA</a>, follow this link to start from the beginning.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>The structure of RNA</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">

<p><b>The function of messenger RNA in the cell</b></p>

<p>You will probably know that the sequence of bases in DNA carries the <b><i>genetic code</i></b>.  Scattered along the DNA molecule are particularly important sequences of bases known as <b><i>genes</i></b>.  Each gene is a coded description for making a particular protein.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>It would be more accurate to say that each gene coded for a particular polypeptide, because some proteins are made of more than one polypeptide chain.    For simplicity, I'm going to refer from now on to the synthesis of a protein, rather than a polypeptide - it sounds less scary!</p>

<p>To be really accurate, some genes code for other sorts of molecule apart from proteins, but we are only going to be looking at the genes involved in protein synthesis.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">
<p>Getting from the code in DNA to the final protein is a very complicated process.</p>

<p>The code is first transcribed (&quot;copied&quot;, although with one important difference  - see later) to messenger RNA.  That then travels out of the nucleus of the cell (where the DNA is found) into the cytoplasm of the cell.  The cytoplasm contains essentially everything else in the cell apart from the nucleus.  Here the code is read and the protein is synthesised with the help of two other forms of RNA - ribosomal RNA and transfer RNA.  We'll talk a lot more about those in a later page.</p>

<p>I'm going to take this complicated process very gently - a bit at a time!</p>
<pre>

</pre>
<p><b>How does messenger RNA differ from DNA?</b></p>

<p>There are several important differences.</p>

<p><b><i>Length</i></b></p>

<p>RNA is much shorter than DNA.  DNA contains the code for making lots and lots of different proteins.  Messenger RNA contains the information to make just one single polypeptide chain - in other words for just one protein, or even just a part of a protein if it is made up of more than one polypeptide chain.</p>

<p><b><i>Overall structure</i></b></p>

<p>DNA has two strands arranged in a double helix.  RNA consists of a single strand.</p>

<p><b><i>The sugar present in the backbone of the chain</i></b></p>

<p>DNA (deoxyribonucleic acid) has a backbone of alternating deoxyribose and phosphate groups.  In RNA (ribonucleic acid), the sugar ribose replaces deoxyribose.</p>

<p>If you have read this sequence of pages from the beginning, you will already have come across the difference between these two sugars.  But to remind you . . .</p>
<pre>

</pre>
<p align="center"><img src="ribvdeoxyrib.gif" width="415" height="191"></p>
<pre>

</pre>
<p>The only difference is the presence of an -OH group on the 2' carbon atom in ribose.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you don't understand what 2' means, you obviously haven't read the first page in this sequence of pages.  It's a bad idea trying to take short cuts with this!<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">
<p><b><i>RNA uses the base uracil (U) rather than thymine (T)</i></b></p>

<p>The structure of uracil is very similar to that of thymine.</p>

<p align="center"><img src="tvu.gif" width="237" height="142"></p>

<p>The nitrogen shown in blue in the uracil is the one which attaches to the 1' carbon in the ribose.  In the process, the hydrogen shown in blue is lost together with the -OH group on the 1' carbon in the ribose.</p>

<p>The only difference between the two molecules is the presence or absence of the CH<sub>3</sub> group.</p>

<p>Uracil can form exactly the same hydrogen bonds with adenine as thymine can - the shape of the two molecules is exactly the same where it matters.</p>

<p>Compare the hydrogen bonding between adenine (A) and thymine (T):</p>

<p align="center"><img src="atbasepair2.gif" width="307" height="200"></p>

<p>. . . with that between adenine (A) and uracil (U):</p>

<p align="center"><img src="aubasepair.gif" width="307" height="185"></p>

<p>In DNA the hydrogen bonding between A and T helps to tie the two strands together into the double helix.  That isn't relevant in RNA because it is only a single strand.  However, you will find several examples in what follows on this and further pages where the ability of adenine (A) to attract and bond with uracil (U) is central to the processes going on.</p>

<p>The base pairing of guanine (G) and cytosine (C) is just the same in DNA and RNA.</p>

<p>So in RNA the important base pairs are:</p>

<ul>
<li><p>adenine (A) pairs with uracil (U);</p></li>

<li><p>guanine (G) pairs with cytosine (C).</p></li>
</ul>
</font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>Transcription</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p>Transcription is the name given to the process where the information in a gene in a DNA strand is transferred to an RNA molecule.</p>
<pre>

</pre>
<p><b>The coding strand and the template strand of DNA</b></p>

<p>The important thing to realise is that the genetic information is carried on only one of the two strands of the DNA.  This is known as the <b><i>coding strand</i></b>.</p>

<p>The other strand is known as the <b><i>template strand</i></b>, for reasons which will become obvious is a moment.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>These two strands are often given other names as well, sometimes in a very confusing way (at least to a non-biochemist!).  The two terms <i>coding</i> and <i>template</i> are commonly used, and seem to me to best describe the function of the two chains.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">
<p><b><i>The coding strand</i></b></p>

<p>The information in a gene on the coding strand is read in the direction from the 5' end to the 3' end.</p>

<p>Remember that the 5' end is the end which has the phosphate group attached to the 5' carbon atom.  The 3' end is the end where the phosphate is attached to a 3' carbon atom - or if it is at the very end of the DNA chain has a free -OH group on the 3' carbon.</p>

<p>You may remember this diagram of a tiny part of a DNA chain from the first page in this sequence:</p>

<p align="center"><img src="dnachain2.gif" width="451" height="339"></p>

<p>If the left-hand chain was the coding chain, the genetic code would be read from the top end (the 5' end) downwards.  The code in this very small fragment of a gene would be read as &quot;. . . A T T G C . . .&quot;.</p>
<pre>

</pre>
<p><b><i>The template strand</i></b></p>

<p>The template strand is <b><i>complementary</i></b> to the coding strand.  That means that every A on the coding strand is matched by a T on the template strand (and vice versa).  Every G on the coding strand is matched by a C on the template strand (and again vice versa).</p>

<p>If you took the template strand and built a new DNA strand on it (as happens in DNA replication), you would get an exact copy of the original DNA coding strand formed.</p>

<p>Almost exactly the same thing happens when you make RNA.  If you build an RNA strand on the template strand, you will get a copy of the information on the DNA coding strand - but with one important difference.</p>

<p>In RNA, uracil (U) is used instead of thymine (T).  So if the original DNA coding strand had the sequence A T T G C T, this would end up in the RNA as A U U G C U - everything is exactly the same except that every T had been replaced by U.</p>
<pre>

</pre>
<p><b>The transcription process</b></p>

<p><b><i>Finding the start of the gene on the coding strand</i></b></p>

<p>Transcription is under the control of the enzyme <b><i>RNA polymerase</i></b>.  The first thing that the enzyme has to do is to find the start of the gene on the coding strand of the DNA.  Remember that DNA has lots of genes strung out along the coding strand.  That means that the  enzyme has to pick the right strand and identify the beginning of each gene.</p>

<p>It does this by recognising and binding with one or more short sequences of bases &quot;upstream&quot; of the start of each gene.  &quot;Upstream&quot; means that it is slightly closer to the 5' end of the DNA strand than the gene.</p>

<p>These base sequences are known as <b><i>promoter sequences</i></b>.</p>

<p align="center"><img src="transcribe1.gif" width="371" height="95"></p>

<p>Remember that the two strands of DNA are hydrogen bonded together.  You can think of the enzyme as being wrapped around both strands.  In fact, the enzyme is big enough to enclose not only the promoter sequence but the beginning of the gene itself.</p>
<pre>

</pre>
<p><b><i>Transcribing the gene and making the RNA</i></b></p>

<p>Once the enzyme has attached to the DNA, it unwinds the double helix over a short length, and splits the two strands apart.  This gives a &quot;bubble&quot; in which the coding strand and template strand are separated over the length of about 10 bases.</p>

<p>The next diagram shows the enzyme in the process of starting to make the new RNA strand.</p>

<p align="center"><img src="transcribe2.gif" width="382" height="254"></p>

<p>New nucleotides are added to the growing RNA chain at the 3' end.  The next nucleotides to be added in the example here would contain the bases G and then C.  The new G in the RNA would complement the C below it in the template strand.  Next after that in the template strand is a G.  That would be complemented by a C in the growing RNA.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>Remember that a nucleotide contains the base attached to a sugar (in this case, ribose) which is attached to a phosphate group.  The ribose and the phosphate add to the backbone of the RNA chain with the bases hanging off that backbone.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">
<p>Now compare the bit of RNA with the coding strand directly above it.  Apart from the fact that every thymine (T) is now a uracil (U) instead, the chains are identical.</p>

<p>Now the enzyme moves along the DNA, zipping it up again behind it.  Essentially it moves the bubble along the chain, adding new nucleotides all the time.  The growing RNA tail becomes detached from the template strand as the enzyme moves along.</p>
<pre>

</pre>
<p align="center"><img src="transcribe3.gif" width="378" height="82"></p>
<pre>

</pre>
<p>How does the enzyme know where to stop after it reaches the end of the gene?  You will remember that it recognises the beginning of the gene by the presence of a promoter sequence of bases upstream of the start.</p>

<p>After the end of the gene (&quot;downstream&quot; of the gene), there will be a <b><i>termination sequence</i></b> of bases.  Once the enzyme gets to those, it stops adding new nucleotides to the chain and detaches the RNA molecule completely from the template chain.</p>
<pre>

</pre>
<p>So . . . we've produced a molecule of messenger RNA - so called because it is now going to carry the genetic code (the message) out of the nucleus of the cell to the cytoplasm where protein synthesis can take place.</p>

<p>Before we look at how that synthesis works, we need to stop and consider the nature of the code itself.  That's on the next page in this sequence.</p>
</font>
<pre>

</pre>
<table align="center" width="100%" cellpadding="5" border="1">
<tr><td bgcolor="#ccffcc"><font color="#000000" face="Helvetica, Arial" size="2">
<p><b>Questions to test your understanding</b></p>

<p>If this is the first set of questions you have done, please read the <a href="../../questions.html">introductory page</a> before you start.  You will need to use the BACK BUTTON on your browser to come back here afterwards.</p>

<p><a href="../questions/q-dna3.pdf">questions on DNA transcription to RNA</a></p>
<p><a href="../questions/a-dna3.pdf">answers</a></p>

</font>
</td></tr>
</table>
<pre>

</pre>
<dl>
<font color="#006600" face="Arial, Helvetica " size="3">
<dt><p><b>Where would you like to go now?</b></p></dt>

<dd><p><a href="dna4.html#top"><b>To the next page about DNA and RNA . . .</b></a></p>
<p><a href="../aminoacidmenu.html#top"><b>To the amino acid and other biochemistry menu . . .</b></a></p>
<p><a href="../../orgpropsmenu.html#top"><b>To the menu of other organic compounds . . .</b></a></p>
<p><a href="../../index.html#top"><b>To Main Menu . . .</b></a></p></dd>
</font></dl>

<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2007 (modified May 2016)</font></p>
</td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacids/dna3/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacids/dna2/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>DNA - replication</title>

<meta name="description" content="A very simplified look at the replication (copying) of DNA">
<meta name="keywords" content="dna, structure, deoxyribonucleic acid, base, nucleotide, double helix, base pair, replicate, replication, copying, copy">

</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p><b>DNA - REPLICATION</b></p></font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="3">
<p>This page takes a very simplified look at how DNA replicates (copies) itself.  It gives only a brief over-view of the process, with no attempt to describe the mechanism.  As such, it is only suitable for 16 - 18 year old <i>chemistry</i> students.  If you are doing biology or biochemistry, you are going to need more detail than this page covers.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you have come straight to this page from a search engine, you should be aware that this is the second page in a sequence of pages about DNA.  Unless you already understand the <a href="dna1.html#top">structure of DNA</a>, follow this link to read that page first.</p>

<p>If you need detailed information about the replication of DNA, a Google search for <b><i>DNA replication</i></b> will lead you to some well thought out and very clear animations of the process.  The best one to start with can be found by following <a href="http://www.wiley.com/college/pratt/0471393878/student/animations/dna_replication/index.html" target="_blank" rel="noopener">this link</a>.  This <a href="http://www.johnkyrk.com/DNAreplication.html" target="_blank" rel="noopener">second link</a> perhaps gives a better view of the over-all process, but is more difficult to understand if you don't look at the other one first.  If you are an A level (or equivalent) chemistry student, only look at these if you are interested, not because you are likely to need it for exam purposes.</p>

<p>Both of these links involve Flash animations.  Unfortunately some devices such as the iPad don't support Flash, and you won't be able to watch these animations on them.  There is no way around that apart from using a standard computer.</p>

<p>I am a bit wary of suggesting links to other sites, because these sites can change.  If you find that either of these links don't work (apart from problems with Flash), please e-mail me via the address on the <a href="../../about.html#top">about this site</a> page.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>Semi-conservative replication</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p><b>A very simple look at the process</b></p>

<p>We'll explain exactly what &quot;semi-conservative&quot; means when we have got some diagrams to look at.  First imagine what happens if the two individual strands in the DNA double helix start to unzip.</p>

<p>The diagram shows this happening <i>in the middle of the DNA double helix</i> - you mustn't assume that the top of the diagram is the end of the chain.  It isn't.  Further up the double helix, the two strands will still be joined together.</p>

<p>In fact, this is happening lots of times along the very long DNA molecule.  Lengths of chain become separated to form what are known as &quot;bubbles&quot;.  If you feel the need to see this in more detail, read the rest of this page, and then have a quick look at the links above.</p> 

<p align="center"><img src="dnarep1.gif" width="316" height="376"></p>

<p>Some of the hydrogen bonds get broken and the two strands become partly separated.</p>

<p>The red dotted lines on the diagram just point out the original base pairs.  These are <b><i>not</i></b> bonds in any form.  These base pairs are now much too far apart for any sort of bonding between them.</p>

<p>Now suppose that you have a source of nucleotides - phosphate joined to deoxyribose joined to a base, including all the four sorts of bases needed for DNA.</p>

<p>The next diagram shows what would happen if a nucleotide containing guanine (G) and one containing cytosine (C) were attracted to the top two bases on the left-hand strand of the unzipped DNA - and then joined together.</p>

<p align="center"><img src="dnarep2.gif" width="316" height="376"></p>

<p>How did they end up joined together?  This is all under the control of a number of enzymes, one of which (DNA polymerase) is responsible for joining up nucleotides along the chain in this way.</p>

<p>Now suppose the same sort of thing happened at the top of the right-hand strand.  You would end up with . . .</p>

<p align="center"><img src="dnarep3.gif" width="316" height="436"></p>

<p>Now compare the double strands that you are forming on the left- and right-hand sides.  They are exactly the same . . . and if you were to continue this process, they would continue to be the same.</p>

<p>And if you compare the patterns of bases in the new DNA being formed with what was in the original DNA before it started to unzip, everything is the same.  This is inevitable because of the way the bases pair together.</p>
<pre>

</pre>
<p><b>What does <i>semi-conservative</i> replication mean?</b></p>

<p>Let's simplify the last diagram, and assume that the whole copying process is complete.  The next diagram focusses on the short bit of the total DNA molecule that we have been looking at.  A typical human DNA molecule is around 150 million base pairs long - you will have to imagine the rest of it!</p>

<p>You have also got to remember that in reality the whole thing would have coiled into its double helix.  Trying to draw that just makes everything look messy and complicated!</p>
<pre>

</pre>
<p align="center"><img src="dnarep4.gif" width="384" height="318"></p>
<pre>

</pre>
<p>The original DNA is shown all in blue.  The red strands in the daughter DNA are the ones which have been built on the original blue strands during the replication process.</p>

<p>You can see that each of the daughter molecules is made of half of the original DNA plus a new strand.  That's all &quot;semi-conservative replication&quot; means.  Half of the original DNA is conserved (kept) in each of the daughter molecules.</p>

<p>The red and blue, of course, have no physical significance apart from as a way of making the diagrams clearer.  All three of these DNA molecules will be identical in every way.</p>
</font>
<pre>

</pre>
<table align="center" width="100%" cellpadding="5" border="1">
<tr><td bgcolor="#ccffcc"><font color="#000000" face="Helvetica, Arial" size="2">
<p><b>Questions to test your understanding</b></p>

<p>If this is the first set of questions you have done, please read the <a href="../../questions.html">introductory page</a> before you start.  You will need to use the BACK BUTTON on your browser to come back here afterwards.</p>

<p><a href="../questions/q-dna2.pdf">questions on DNA replication</a></p>
<p><a href="../questions/a-dna2.pdf">answers</a></p>

</font>
</td></tr>
</table>
<pre>

</pre>
<dl>
<font color="#006600" face="Arial, Helvetica " size="3">
<dt><p><b>Where would you like to go now?</b></p></dt>


<dd><p><a href="dna3.html#top"><b>To the next page about DNA . . .</b></a></p>
<p><a href="../aminoacidmenu.html#top"><b>To the amino acid and other biochemistry menu . . .</b></a></p>
<p><a href="../../orgpropsmenu.html#top"><b>To the menu of other organic compounds . . .</b></a></p>
<p><a href="../../index.html#top"><b>To Main Menu . . .</b></a></p></dd>
</font></dl>

<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2007 (modified May 2016)</font></p>
</td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacids/dna2/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacids/dna1/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>DNA - structure</title>

<meta name="description" content="A fairly detailed look at the structure of DNA">
<meta name="keywords" content="dna, structure, deoxyribonucleic acid, sugar, deoxyribose, ribose, phosphate, base, nucleotide, adenine, guanine, thymine, cytosine, double helix, hydrogen bond, base pair">

</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p><b>DNA - STRUCTURE</b></p></font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="3">
<p>This page, looking at the structure of DNA, is the first in a sequence of pages leading on to how DNA replicates (makes copies of) itself, and then to how information stored in DNA is used to make protein molecules.  This material is aimed at 16 - 18 year old <i>chemistry</i> students.  If you are interested in this from a biological or biochemical point of view, you may find these pages a useful introduction before you get more information somewhere else.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you are doing biology or biochemistry and are interested in more detail you can download a very useful <a href="http://www.biochemistry.org/Portals/0/Education/Docs/BASC02_full.pdf" target="_blank" rel="noopener">pdf file about DNA</a> from the Biochemical Society.</p>

<p>Chemistry students at UK A level (or its various equivalents) should <b><i>not</i></b> waste time on this.  The booklet is written for A level biology students, and goes into far more detail than you will need for chemistry purposes.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>A quick look at the whole structure of DNA</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p>These days, most people know about DNA as a complex molecule which carries the genetic code.  Most will also have heard of the famous double helix.</p>

<p>I'm going to start with a diagram of the whole structure, and then take it apart to see how it all fits together.  The diagram shows a tiny bit of a DNA double helix.</p>

<p align="center"><img src="doublehelix.gif" width="356" height="367"></p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>This diagram comes from the <a href="http://ghr.nlm.nih.gov/handbook/basics/dna" target="_blank" rel="noopener">US National Library of Medicine</a>.  You can see it in its original context by following this link if you are interested.</p>

<p>Normally I prefer to draw my own diagrams, but my drawing software isn't sophisticated enough to produce convincing twisted &quot;ribbons&quot;.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>Exploring a DNA chain</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p><b>The sugars in the backbone</b></p>

<p>The backbone of DNA is based on a repeated pattern of a sugar group and a phosphate group.  The full name of DNA, deoxyribonucleic acid, gives you the name of the sugar present - deoxyribose.</p>

<p>Deoxyribose is a modified form of another sugar called ribose.  I'm going to give you the structure of that first, because you will need it later anyway.  Ribose is the sugar in the backbone of RNA, ribonucleic acid.</p>

<p align="center"><img src="ribose.gif" width="104" height="113"></p>

<p>This diagram misses out the carbon atoms in the ring for clarity.  Each of the four corners where there isn't an atom shown has a carbon atom.</p>

<p>The heavier lines are coming out of the screen or paper towards you.  In other words, you are looking at the molecule from a bit above the plane of the ring.</p>

<p>So that's ribose.  Deoxyribose, as the name might suggest, is ribose which has lost an oxygen atom - &quot;de-oxy&quot;.</p>

<p align="center"><img src="deoxyribose.gif" width="263" height="174"></p>

<p>The only other thing you need to know about deoxyribose (or ribose, for that matter) is how the carbon atoms in the ring are numbered.</p>

<p>The carbon atom to the right of the oxygen as we have drawn the ring is given the number 1, and then you work around to the carbon on the CH<sub>2</sub>OH side group which is number 5.</p>

<p align="center"><img src="deoxyriboseno.gif" width="377" height="149"></p>

<p>You will notice that each of the numbers has a small dash by it - 3' or 5', for example.  If you just had ribose or deoxyribose on its own, that wouldn't be necessary, but in DNA and RNA these sugars are attached to other ring compounds.  The carbons in the sugars are given the little dashes so that they can be distinguished from any numbers given to atoms in the other rings.</p>

<p>You read 3' or 5' as &quot;3-prime&quot; or &quot;5-prime&quot;.</p>
<pre>

</pre>
<p><b>Attaching a phosphate group</b></p>

<p>The other repeating part of the DNA backbone is a phosphate group.  A phosphate group is attached to the sugar molecule in place of the -OH group on the 5' carbon.</p>

<p align="center"><img src="phossugar.gif" width="136" height="134"></p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>You may find other versions of this with varying degrees of ionisation.  You may find a hydrogen attached instead of having a negative charge on one of the oxygens, or the hydrogen removed from the top -OH group to leave a negative ion there as well.</p>

<p>I don't want to get bogged down in this.  The version I am using is fine for chemistry purposes, and will make it easy to see how the DNA backbone is put together.  We are soon going to simplify all this down anyway!<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p><b>Attaching a base and making a <i>nucleotide</i></b></p>

<p>The final piece that we need to add to this structure before we can build a DNA strand is one of four complicated organic bases.  In DNA, these bases are <b><i>cytosine (C)</i></b>, <b><i>thymine (T)</i></b>, <b><i>adenine (A)</i></b> and <b><i>guanine (G)</i></b>.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>These are called &quot;bases&quot; because that is exactly what they are in chemical terms.  They have lone pairs on nitrogens and so can act as electron pair donors (or accept hydrogen ions, if you prefer the simpler definition).  This isn't particularly relevant to their function in DNA, but they are always referred to as bases anyway.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p>These bases attach in place of the -OH group on the 1' carbon atom in the sugar ring.</p>

<p align="center"><img src="nucleotide1.gif" width="173" height="134"></p>

<p>What we have produced is known as a <b><i>nucleotide</i></b>.</p>

<p>We now need a quick look at the four bases.  If you need these in a chemistry exam at this level, the structures will almost certainly be given to you.</p>

<p>Here are their structures:</p>

<p align="center"><img src="ctag.gif" width="290" height="331"></p>

<p>The nitrogen and hydrogen atoms shown in blue on each molecule show where these molecules join on to the deoxyribose.  In each case, the hydrogen is lost together with the -OH group on the 1' carbon atom of the sugar.  This is a condensation reaction - two molecules joining together with the loss of a small one (not necessarily water).</p>

<p>For example, here is what the nucleotide containing cytosine would look like:</p>

<p align="center"><img src="nucleotide2.gif" width="170" height="147"></p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>I've flipped the cytosine horizontally (compared with the structure of cytosine I've given previously) so that it fits better into the diagram.  You must be prepared to rotate or flip these structures if necessary.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p><b>Joining the nucleotides into a DNA strand</b></p>

<p>A DNA strand is simply a string of nucleotides joined together.  I can show how this happens perfectly well by going back to a simpler diagram and not worrying about the structure of the bases.</p>

<p>The phosphate group on one nucleotide links to the 3' carbon atom on the sugar of another one.  In the process, a molecule of water is lost - another condensation reaction.</p>

<p align="center"><img src="makedna.gif" width="436" height="293"></p>

<p>. . . and you can continue to add more nucleotides in the same way to build up the DNA chain.</p>

<p>Now we can simplify all this down to the bare essentials!</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>You will notice that I have drawn the P-O bonds attaching to the two sugar molecules opposite each other in the diagram above.  You will also find diagrams where they are drawn at right angles to each other.  Which is right?</p>

<p>Both are right and, equally, both are misleading!  The shape of the bonds around the phosphorus atom is tetrahedral, and all of the bonds are at approximately 109&deg; to each other.  Whichever way you choose to draw this in 2-dimensions on paper, it still represents the same molecule in reality.</p>

<p>To take a simpler example, if you draw a structural formula for CH<sub>2</sub>Cl<sub>2</sub> using simple bond notation, you could equally well draw the chlorine atoms at right angles to each other or opposite each other.  The molecule would still be exactly the same.  This is one of the things you had to learn when you first started drawing structures for organic molecules.  If you still aren't sure about this, look again at the page about <a href="../../basicorg/conventions/draw.html">drawing organic molecules</a>.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">
<pre>

</pre>
<p><b>Building a DNA chain concentrating on the essentials</b></p>

<p>What matters in DNA is the sequence the four bases take up in the chain.  We aren't particularly interested in the backbone, so we can simplify that down.  For the moment, we can simplify the precise structures of the bases as well.</p>

<p>We can build the chain based on this fairly obvious simplification:</p>

<p align="center"><img src="nucleotide3.gif" width="150" height="52"></p>

<p>There is only one possible point of confusion here - and that relates to how the phosphate group, <b>P</b>, is attached to the sugar ring.  Notice that it is joined via <i>two</i> lines with an angle between them.</p>

<p>By convention, if you draw lines like this, there is a carbon atom where these two lines join.  That is the carbon atom in the CH<sub>2</sub> group if you refer back to a previous diagram.  If you had tried to attach the phosphate to the ring by a single straight line, that CH<sub>2</sub> group would have got lost!</p>

<p>Joining up lots of these gives you a part of a DNA chain.  The diagram below is a bit from the middle of a chain.  Notice that the individual bases have been identified by the first letters of the base names.  (A  = adenine, etc).  Notice also that there are two different sizes of base.  Adenine and guanine are bigger because they both have two rings.  Cytosine and thymine only have one ring each.</p>

<p align="center"><img src="dnachain.gif" width="323" height="400"></p>

<p>If the top of this segment was the end of the chain, then the phosphate group would have an -OH group attached to the spare bond rather than another sugar ring.</p>

<p>Similarly, if the bottom of this segment of chain was the end, then the spare bond at the bottom would also be to an -OH group on the deoxyribose ring.</p>
</font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>Joining the two DNA chains together</b></p>
</font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p><b>The importance of &quot;base pairs&quot;</b></p>

<p>Have another look at the diagram we started from:</p>

<p align="center"><img src="doublehelix.gif" width="356" height="367"></p>

<p>If you look at this carefully, you will see that an adenine on one chain is always paired with a thymine on the second chain.  And a guanine on one chain is always paired with a cytosine on the other one.</p>

<p>So how exactly does this work?</p>

<p>The first thing to notice is that a smaller base is always paired with a bigger one.  The effect of this is to keep the two chains at a fixed distance from each other all the way along.</p>

<p>But, more than this, the pairing has to be <i>exactly</i> . . .</p>

<ul>
<li><p>adenine (A) pairs with thymine (T);</p></li>

<li><p>guanine (G) pairs with cytosine (C).</p></li>
</ul>

<p>That is because these particular pairs fit exactly to form very effective hydrogen bonds with each other.  It is these hydrogen bonds which hold the two chains together.</p>

<p>The base pairs fit together as follows.</p>

<p>The A-T base pair:</p>

<p align="center"><img src="atbasepair.gif" width="307" height="200"></p>

<p>The G-C base pair:</p>

<p align="center"><img src="gcbasepair.gif" width="349" height="263"></p>
<pre>

</pre>
<p>If you try any other combination of base pairs, they won't fit!</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If the structures confuse you at first sight, it is because the molecules have had to be turned around from the way they have been drawn above in order to make them fit.  Be sure that you understand how to do that.  As long as you were given the structures of the bases, you could be asked to show how they hydrogen bond - and that would include showing the lone pairs and polarity of the important atoms.</p>

<p>If <a href="../../atoms/bonding/hbond.html">hydrogen bonding</a> worries you, follow this link for detailed explanations.  Use the BACK button on your browser to return here later.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p><b>A final structure for DNA showing the important bits</b></p>

<p align="center"><img src="dnachain2.gif" width="451" height="339"></p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>You might have noticed that I have shorten the chains by one base pair compared with the previous diagram.  There isn't any sophisticated reason for this.  The diagram just got a little bit too big for my normal page width, and it was a lot easier to just chop a bit off the bottom than rework all my previous diagrams to make them slightly smaller!  This diagram only represents a tiny bit of a DNA molecule anyway.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="3">

<p>Notice that the two chains run in opposite directions, and the right-hand chain is essentially upside-down.  You will also notice that I have labelled the ends of these bits of chain with 3' and 5'.</p>

<p>If you followed the left-hand chain to its very end at the top, you would have a phosphate group attached to the 5' carbon in the deoxyribose ring.  If you followed it all the way to the other end, you would have an -OH group attached to the 3' carbon.</p>

<p>In the second chain, the top end has a 3' carbon, and the bottom end a 5'.</p>

<p>This 5' and 3' notation becomes important when we start talking about the genetic code and genes.  The genetic code in genes is always written in the 5' to 3' direction along a chain.</p>

<p>It is also important when we take a very simplified look at how DNA makes copies of itself on the next page . . .</p>
</font>
<pre>

</pre>
<table align="center" width="100%" cellpadding="5" border="1">
<tr><td bgcolor="#ccffcc"><font color="#000000" face="Helvetica, Arial" size="2">
<p><b>Questions to test your understanding</b></p>

<p>If this is the first set of questions you have done, please read the <a href="../../questions.html">introductory page</a> before you start.  You will need to use the BACK BUTTON on your browser to come back here afterwards.</p>

<p><a href="../questions/q-dna1.pdf">questions on the structure of DNA</a></p>
<p><a href="../questions/a-dna1.pdf">answers</a></p>

</font>
</td></tr>
</table>
<pre>

</pre>
<dl>
<font color="#006600" face="Arial, Helvetica " size="3">
<dt><p><b>Where would you like to go now?</b></p></dt>


<dd><p><a href="dna2.html#top"><b>To the next page about DNA . . .</b></a></p>
<p><a href="../aminoacidmenu.html#top"><b>To the amino acid and other biochemistry menu . . .</b></a></p>
<p><a href="../../orgpropsmenu.html#top"><b>To the menu of other organic compounds . . .</b></a></p>
<p><a href="../../index.html#top"><b>To Main Menu . . .</b></a></p></dd>
</font></dl>

<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2007 (modified May 2016)</font></p>
</td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacids/dna1/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacids/background/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>an introduction to amino acids</title>

<meta name="description" content="An introduction to amino acids, including their physical properties">
<meta name="keywords" content="amino, acid, acids, physical, properties, solubility, boiling points">

</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p><b>INTRODUCING AMINO ACIDS</b></p></font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="3">
<p>This page explains what amino acids are, concentrating on the 2-amino acids that are biologically important.  It looks in some detail at their simple physical properties such as solubility and melting points.</p>
</font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>What are amino acids?</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">

<p><b>Structures and names</b></p>

<p>Amino acids are exactly what they say they are!  They are compounds containing an amino group, -NH<sub>2</sub>, and a carboxylic acid group, -COOH.</p>

<p>The biologically important amino acids have the amino group attached to the carbon atom next door to the -COOH group.  They are known as <b><i>2-amino acids</i></b>.  They are also known (slightly confusingly) as <b><i>alpha-amino acids</i></b>.  These are the ones we will concentrate on.</p>

<p>The two simplest of these amino acids are 2-aminoethanoic acid and 2-aminopropanoic acid.</p>

<p>Because of the biological importance of molecules like these, they are normally known by their traditional biochemical names.</p>

<p>2-aminoethanoic acid, for example, is usually called <b><i>glycine</i></b>, and 2-aminopropanoic acid is usually known as <b><i>alanine</i></b>.</p>

<p align="center"><img src="aafirst2.gif" width="296" height="94"></p>
<pre>

</pre>
<p>The general formula for a 2-amino acid is:</p>

<p align="center"><img src="aagenform.gif" width="75" height="36"></p>

<p>. . . where &quot;R&quot; can be quite a complicated group containing other active groups like -OH, -SH, other amine or carboxylic acid groups, and so on.  It is definitely NOT necessarily a simple hydrocarbon group!</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>For complete accuracy, one of the 20 biologically important amino acids (proline) has a slightly different structure.  The &quot;R&quot; group is bent into a circle which attaches itself to the nitrogen again in place of one of the hydrogens.  This complication doesn't actually make much difference to the chemistry of the compound - the nitrogen still behaves in the same way as it does in the other amino acids.  This isn't something you need to worry about for chemistry purposes at this introductory level.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>Physical properties</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">

<p><b>Melting points</b></p>

<p>The amino acids are crystalline solids with surprisingly high melting points.  It is difficult to pin the melting points down exactly because the amino acids tend to decompose before they melt.  Decomposition and melting tend to be in the 200 - 300&deg;C range.</p>

<p>For the size of the molecules, this is very high.  Something unusual must be happening.</p>

<p>If you look again at the general structure of an amino acid, you will see that it has both a basic amine group and an acidic carboxylic acid group.</p>

<p align="center"><img src="aaacidbase.gif" width="110" height="149"></p>

<p>There is an internal transfer of a hydrogen ion from the -COOH group to the -NH<sub>2</sub> group to leave an ion with both a negative charge and a positive charge.</p>

<p>This is called a <b><i>zwitterion</i></b>.</p>

<p align="center"><img src="zwitterion.gif" width="77" height="71"></p>

<p>A zwitterion is a compound with no overall electrical charge, but which contains separate parts which are positively and negatively charged.</p>

<p>This is the form that amino acids exist in even in the solid state.  Instead of the weaker hydrogen bonds and other intermolecular forces that you might have expected, you actually have much stronger ionic attractions between one ion and its neighbours.</p>

<p>These ionic attractions take more energy to break and so the amino acids have high melting points for the size of the molecules.</p>
<pre>

</pre>
<p><b>Solubility</b></p>

<p>Amino acids are generally soluble in water and insoluble in non-polar organic solvents such as hydrocarbons.</p>

<p>This again reflects the presence of the zwitterions.  In water, the ionic attractions between the ions in the solid amino acid are replaced by strong attractions between polar water molecules and the zwitterions.  This is much the same as any other ionic substance dissolving in water.</p>

<p>The extent of the solubility in water varies depending on the size and nature of the &quot;R&quot; group.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>At this point I would normally try to relate the actual values for solubility of the various amino acids to their structures.  Unfortunately, from the solubility values that I have got (and I'm not convinced they are necessarily right), I can't find any obvious patterns - in fact, there are some very strange cases indeed.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">
<p>The lack of solubility in non-polar organic solvents such as hydrocarbons is because of the lack of attraction between the solvent molecules and the zwitterions.  Without strong attractions between solvent and amino acid, there won't be enough energy released to pull the ionic lattice apart.</p>
<pre>

</pre>
<p><b>Optical activity</b></p>

<p>If you look yet again at the general formula for an amino acid, you will see that (apart from glycine, 2-aminoethanoic acid) the carbon at the centre of the structure has four different groups attached.  In glycine, the &quot;R&quot; group is another hydrogen atom.</p>

<p align="center"><img src="chiral1.gif" width="153" height="111"></p>

<p>This is equally true if you draw the structure of the zwitterion instead of this simpler structure.</p>

<p>Because of these four different groups attached to the same carbon atom, amino acids (apart from glycine) are <b><i>chiral</i></b>.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#ff0000" face="Helvetica, Arial" size="2">
<p><hr><b>Important: &nbsp;</b>If you don't know <i>exactly</i> what that means, follow this link to the page about <a href="../../basicorg/isomerism/optical.html#top">optical isomerism</a>.  You will find the optical activity of amino acids discussed at the bottom of that page, but read the whole page to be sure that you understand what is going on.</p>

<p>Use the BACK button on your browser to return to this page.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">
<p>The lack of a plane of symmetry means that there will be two stereoisomers of an amino acid (apart from glycine) - one the non-superimposable mirror image of the other.</p>

<p>For a general 2-amino acid, the isomers are:</p>

<p align="center"><img src="chiral2.gif" width="305" height="157"></p>

</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>If you don't know what the various bond symbols mean, you shouldn't have got this far!  Follow the link mentioned above to the page about <a href="../../basicorg/isomerism/optical.html#top">optical isomerism</a>.  Read that page and follow the further link on that page to drawing organic molecules.</p>

<p>Use the BACK button on your browser to return to this page.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">

<p>All the naturally occurring amino acids have the right-hand structure in this diagram.  This is known as the &quot;L-&quot; configuration.  The other one is known as the &quot;D-&quot; configuration.</p>

<p>You almost certainly don't need to know this for UK A level chemistry purposes, but if you are interested, you can recognise the L- configuration by imagining that you are looking down from above on the right-hand structure in the last diagram - <i>in other words, with the hydrogen atom closest to you</i>.  If you read around the other groups in a clockwise direction, you get the word CORN.</p>

<p align="center"><img src="chiral3.gif" width="316" height="182"></p>

</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Warning: &nbsp;</b>There are various other ways of working this out also based on the word CORN, but by looking at the molecule from a different viewpoint which could mean that CORN has to be applied anti-clockwise rather than clockwise for the L- form.</p>

<p>If you have already learnt a different rule, then stick to it.  If you are an A level (or equivalent) chemistry student, find out what (if anything) your examiners expect, and learn that.  If you don't need to know about it, forget it!<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">
<p>You can't tell by looking at a structure whether that isomer will rotate the plane of polarisation of plane polarised light clockwise or anticlockwise.  All the naturally occurring amino acids have the same L- configuration, but they include examples which rotate the plane clockwise (+)  and those which do the opposite (-).</p>

<p>For example:</p>

<ul>
<li><p>(+)alanine</p></li>

<li><p>(-)cysteine</p></li> 

<li><p>(-)tyrosine</p></li> 

<li><p>(+)valine</p></li>
</ul> 

<p>It is quite common for natural systems to only work with one of the optical isomers (enantiomers) of an optically active substance like the amino acids.  It isn't too difficult to see why that might be.  Because the molecules have different spatial arrangements of their various groups, only one of them is likely to fit properly into the active sites on the enzymes they work with.</p>
</font>
<pre>

</pre>
<table align="center" width="100%" cellpadding="5" border="1">
<tr><td bgcolor="#ccffcc"><font color="#000000" face="Helvetica, Arial" size="2">
<p><b>Questions to test your understanding</b></p>

<p>If this is the first set of questions you have done, please read the <a href="../../questions.html">introductory page</a> before you start.  You will need to use the BACK BUTTON on your browser to come back here afterwards.</p>

<p><a href="../questions/q-aaintro.pdf">questions on the introduction to amino acids</a></p>
<p><a href="../questions/a-aaintro.pdf">answers</a></p>

</font>
</td></tr>
</table>
<pre>

</pre>
<dl>
<font color="#006600" face="Arial, Helvetica " size="3">
<dt><p><b>Where would you like to go now?</b></p></dt>

<dd><p><a href="../aminoacidmenu.html#top"><b>To the amino acid and proteins menu . . .</b></a></p>
<p><a href="../../orgpropsmenu.html#top"><b>To the menu of other organic compounds . . .</b></a></p>
<p><a href="../../index.html#top"><b>To Main Menu . . .</b></a></p></dd>
</font></dl>

<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2004 (last modified April 2016)</font></p>
</td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacids/background/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacids/acidbase/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>the acid base behaviour of amino acids</title>

<meta name="description" content="The acid-base behaviour of amino acids">
<meta name="keywords" content="amino, acid, acids, zwitterion, zwitterions, base">

</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p><b>THE ACID-BASE BEHAVIOUR OF AMINO ACIDS</b></p></font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="3">
<p>This page looks at what happens to amino acids as you change the pH by adding either acids or alkalis to their solutions.</p>

<p>For simplicity, the page only looks at amino acids which contain a single -NH<sub>2</sub> group and a single -COOH group.</p>
</font>
<pre>

</pre>
<font color="#000000" face="Helvetica, Arial" size="4">
<p><b>Amino acids as zwitterions</b></p></font>

<font color="#000000" face="Helvetica, Arial" size="3">
<p><b>Zwitterions in simple amino acid solutions</b></p>

<p>An amino acid has both a basic amine group and an acidic carboxylic acid group.</p>

<p align="center"><img src="aaacidbase.gif" width="110" height="149"></p>

<p>There is an internal transfer of a hydrogen ion from the -COOH group to the -NH<sub>2</sub> group to leave an ion with both a negative charge and a positive charge.</p>

<p>This is called a <b><i>zwitterion</i></b>.</p>

<p align="center"><img src="zwitterion.gif" width="77" height="71"></p>

<p>This is the form that amino acids exist in even in the solid state.  If you dissolve the amino acid in water, a simple solution also contains this ion.</p>

<p>A zwitterion is a compound with no overall electrical charge, but which contains separate parts which are positively and negatively charged.</p>
<pre>

</pre>
<p><b>Adding an alkali to an amino acid solution</b></p>

<p>If you increase the pH of a solution of an amino acid by adding hydroxide ions, the hydrogen ion is removed from the -NH<sub>3</sub><sup>+</sup> group.</p>

<p><img src="padding.gif" width="40" height="15"><img src="zwittbase.gif" width="377" height="140"></p>

<p>You could show that the amino acid now existed as a negative ion using <b><i>electrophoresis</i></b>.</p>

<p>In its simplest form, electrophoresis can just consist of a piece of moistened filter paper on a microscope slide with a crocodile clip at each end attached to a battery.  A drop of amino acid solution is placed in the centre of the paper.</p>

<p>Although the amino acid solution is colourless, its position after a time can be found by spraying it with a solution of <b><i>ninhydrin</i></b>.  If the paper is allowed to dry and then heated gently, the amino acid shows up as a coloured spot.</p>

<p>The amino acid would be found to travel towards the anode (the positive electrode).</p>
<pre>

</pre>
<p><b>Adding an acid to an amino acid solution</b></p>

<p>If you decrease the pH by adding an acid to a solution of an amino acid, the -COO<sup>-</sup> part of the zwitterion picks up a hydrogen ion.</p>

<p><img src="padding.gif" width="40" height="15"><img src="zwittacid.gif" width="410" height="140"></p>

<p>This time, during electrophoresis, the amino acid would move towards the cathode (the negative electrode).</p>
<pre>

</pre>
<p><b>Shifting the pH from one extreme to the other</b></p>

<p>Suppose you start with the ion we've just produced under acidic conditions and slowly add alkali to it.</p>

<p>That ion contains two acidic hydrogens - the one in the -COOH group and the one in the -NH<sub>3</sub><sup>+</sup> group.</p>

<p>The more acidic of these is the one in the -COOH group, and so that is removed first - and you get back to the zwitterion.</p>

<p><img src="padding.gif" width="40" height="15"><img src="zwittatob1.gif" width="387" height="44"></p>

<p>So when you have added just the right amount of alkali, the amino acid no longer has a net positive or negative charge. That means that it wouldn't move towards either the cathode or anode during electrophoresis.</p>

<p>The pH at which this lack of movement during electrophoresis happens is known as the <b><i>isoelectric point</i></b> of the amino acid.  This pH varies from amino acid to amino acid.</p>

<p>If you go on adding hydroxide ions, you will get the reaction we've already seen, in which a hydrogen ion is removed from the -NH<sub>3</sub><sup>+</sup> group.</p>

<p><img src="padding.gif" width="40" height="15"><img src="zwittatob2.gif" width="349" height="41"></p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>You might have expected the isoelectric point to be at pH 7 - when the solution is neither acidic nor alkaline.  In fact, the isoelectric point for many amino acids is about pH 6.  Explaining why it isn't at pH 7 is quite long-winded and almost certainly beyond what you will need for UK A level (or its equivalent) purposes.  If you are interested, the problem is discussed at the bottom of this page.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">
<p>You can, of course, reverse the whole process by adding an acid to the ion we've just finished up with.</p>

<p>That ion contains two basic groups - the -NH<sub>2</sub> group and the -COO<sup>-</sup> group.  The -NH<sub>2</sub> group is the stronger base, and so picks up hydrogen ions first.  That leads you back to the zwitterion again.</p>

<p><img src="padding.gif" width="40" height="15"><img src="zwittbtoa1.gif" width="320" height="41"></p>

<p>. . . and, of course, you can keep going by then adding a hydrogen ion to the -COO<sup>-</sup> group.</p>

<p><img src="padding.gif" width="40" height="15"><img src="zwittbtoa2.gif" width="350" height="44"></p>
<pre>

</pre>
<p><b>Why isn't the isoelectric point of an amino acid at pH 7?</b></p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#ff0000" face="Helvetica, Arial" size="2">
<p><hr><b>Warning: &nbsp;</b>If you are a studying a UK-based syllabus, you are extremely unlikely to need this for exam purposes.  It is included only for interest - you can probably safely ignore it.  Check your <a href="../../syllabuses.html#top">syllabus and past papers</a>.  If you haven't got these, follow this link to find out how to get hold of them.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">

<font color="#000000" face="Helvetica, Arial" size="3">
<p>When an amino acid dissolves in water, the situation is a little bit more complicated than we tend to pretend at this level.  The zwitterion interacts with water molecules - acting as both an acid and a base.</p>

<p>As an acid:</p>

<p><img src="padding.gif" width="40" height="15"><img src="watereqm1.gif" width="372" height="44"></p>

<p>The -NH<sub>3</sub><sup>+</sup> group is a weak acid and donates a hydrogen ion to a water molecule.  Because it is only a weak acid, the position of equilibrium will lie to the left.</p>

<p>As a base:</p>

<p><img src="padding.gif" width="40" height="15"><img src="watereqm2.gif" width="369" height="44"></p>

<p>The -COO<sup>-</sup> group is a weak base and takes a hydrogen ion from a water molecule.  Again, the equilibrium lies to the left.</p>

<p>When you dissolve an amino acid in water, both of these reactions are happening.</p>

<p>But . . . </p>

<p>The positions of the two equilibria aren't identical - they vary depending on the influence of  the &quot;R&quot; group.  In practice, for the simple amino acids we have been talking about, the position of the first equilibrium lies a bit further to the right than the second one.</p>

<p>That means that there will be rather more of the negative ion from the amino acid in the solution than the positive one.</p>

<p>In those circumstances, if you carried out electrophoresis on the unmodified solution, there would be a slight drift of amino acid towards the positive electrode (the anode).</p>

<p>To stop that, you need to cut down the amount of the negative ion so that the concentrations of the two ions are identical.  You can do that by adding a very small amount of acid to the solution, moving the position of the first equilibrium further to the left.</p>

<p>Typically, the pH has to be lowered to about 6 to achieve this.  For glycine, for example, the isoelectric point is pH 6.07; for alanine, 6.11; and for serine, 5.68.</p>
</font>
</td></tr>

<tr><td width="90"></td>
<td><font color="#006600" face="Helvetica, Arial" size="2">
<p><hr><b>Note: &nbsp;</b>I need to stress again that these figures (and the arguments) only hold where there is only one -NH<sub>2</sub> group and one -COOH group in the amino acid.  The isoelectric points are quite different if the molecule contains a second -NH<sub>2</sub> or -COOH group.<br>
</p></font>
<hr><br>
<p></p>
</td></tr>

<tr>
<td colspan="2">
<pre>

</pre>
<table align="center" width="100%" cellpadding="5" border="1">
<tr><td bgcolor="#ccffcc"><font color="#000000" face="Helvetica, Arial" size="2">
<p><b>Questions to test your understanding</b></p>

<p>If this is the first set of questions you have done, please read the <a href="../../questions.html">introductory page</a> before you start.  You will need to use the BACK BUTTON on your browser to come back here afterwards.</p>

<p><a href="../questions/q-aaacidbase.pdf">questions on the acid-base properties of amino acids</a></p>
<p><a href="../questions/a-aaacidbase.pdf">answers</a></p>

</font>
</td></tr>
</table>
<pre>

</pre>
<dl>
<font color="#006600" face="Arial, Helvetica " size="3">
<dt><p><b>Where would you like to go now?</b></p></dt>

<dd><p><a href="../aminoacidmenu.html#top"><b>To the amino acid and proteins menu . . .</b></a></p>
<p><a href="../../orgpropsmenu.html#top"><b>To the menu of other organic compounds . . .</b></a></p>
<p><a href="../../index.html#top"><b>To Main Menu . . .</b></a></p></dd>
</font></dl>

<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2004 (modified April 2016)</font></p>
</td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacids/acidbase/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/organicprops/aminoacidmenu/" ></a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  
<html>
<head><meta name="generator" content="Hexo 3.9.0">
<title>amino acids and proteins menu</title>
</head>

<body bgcolor="#ffffcc" link="blue" vlink="teal" alink="red">
<a name="top"></a>
<center>
<table align="center" border="0" width="480" cellspacing="10">

<tr>
<td colspan="2" bgcolor="#ccffcc" height="50" align="center" valign="middle">
<font color="#006600" size="6" face="Helvetica, Arial"><b>Understanding Chemistry</b></font>
</td></tr>

<tr>
<td colspan="2"><font color="#000000" size="5" face="Helvetica, Arial">
<p>&nbsp;</p>
<p><b>AMINO ACIDS and OTHER BIOCHEMISTRY MENU</b></p></font>

<font color="#000000" size="4" face="Helvetica, Arial">
<p>&nbsp;</p>

<p><b>Amino acids</b></p>
</font>
<font color="#000000" size="3" face="Helvetica, Arial">
<dl>
<dt><p><a href="aminoacids/background.html#top"><b>Background . . .</b></a></p></dt>
<dd><p>An introduction to amino acids including their physical properties.</p></dd>

<dt><p><a href="aminoacids/acidbase.html#top"><b>Acid-base reactions of amino acids . . .</b></a></p></dt>
<dd><p>Amino acids as zwitterions and the influence of this on their reactions with acids and bases.</p></dd>
</dl>
</font>

<font color="#000000" size="4" face="Helvetica, Arial">
<pre>

</pre>
<p><b>Proteins</b></p>
</font>
<font color="#000000" size="3" face="Helvetica, Arial">
<dl>
<dt><p><a href="aminoacids/proteinstruct.html#top"><b>The structure of proteins . . .</b></a></p></dt>
<dd><p>A brief introduction to protein structure including primary, secondary and tertiary structure.</p></dd>

<dt><p><a href="aminoacids/proteinhydrolysis.html#top"><b>Hydrolysis of proteins . . .</b></a></p></dt>
<dd><p>Hydrolysing proteins using hydrochloric acid.</p></dd>
<dt><p><a href="aminoacids/enzymes.html#top"><b>Proteins as enzymes . . .</b></a></p></dt>
<dd><p>A simple introduction to how enzymes act as catalysts.  The page assumes a knowledge of protein structure.</p></dd>

<dt><p><a href="aminoacids/enzymes2.html#top"><b>The effect of changing conditions on enzyme catalysis . . .</b></a></p></dt>
<dd><p>An explanation of the effect of substrate concentration, temperature and pH on enzymes.  This page follows on from the &quot;Proteins as enzymes&quot; page.</p></dd>

<dt><p><a href="aminoacids/enzymes3.html#top"><b>Enzyme inhibitors . . .</b></a></p></dt>
<dd><p>A look at various ways in which enzymes can be prevented from catalysing their reactions.  This also follows on from the &quot;Proteins as enzymes&quot; page.</p></dd>
</dl>
</font>
<pre>

</pre>
<font color="#000000" size="4" face="Helvetica, Arial">

<p><b>DNA</b></p>
</font>
<font color="#000000" size="3" face="Helvetica, Arial">

<p>These pages are designed to be read in sequence.  Later pages will assume knowledge of what has gone before.</p>

<p><a href="aminoacids/dna1.html#top"><b>The structure of DNA . . .</b></a></p>

<p><a href="aminoacids/dna2.html#top"><b>The replication of DNA . . .</b></a></p>

<p><a href="aminoacids/dna3.html#top"><b>The transcription of DNA into messenger RNA . . .</b></a></p>

<p><a href="aminoacids/dna4.html#top"><b>The genetic code . . .</b></a></p>

<p><a href="aminoacids/dna5.html#top"><b>Protein synthesis . . .</b></a></p>

<p><a href="aminoacids/dna6.html#top"><b>DNA mutations . . .</b></a></p>
<pre>


</pre>
<font color="#000000" size="3" face="Helvetica, Arial">

<p><a href="../orgpropsmenu.html#top"><b>Go to menu of other organic compounds . . .</b></a></p>
<p><a href="../index.html#top"><b>Go to Main Menu . . .</b></a></p>
</font>
<pre>

</pre>
<hr>

<p><font color="#000000" size="2" face="Helvetica, Arial"> &copy; Jim Clark 2004 (major additions 2007)</font></p>
</font></td>
</tr>

</table></center>
</body>
</html>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/organicprops/aminoacidmenu/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-12-05 </div>
			<div class="article-title"><a href="/gene-science/" >Genetics Science</a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  <p><a href="https://ghr.nlm.nih.gov/" target="_blank" rel="noopener">Genetics Home Reference - NIH</a></p>
<p><a href="https://ghr.nlm.nih.gov/primer" target="_blank" rel="noopener">Help Me Understand Genetics - Genetics Home Reference - NIH</a></p>
<h2 id="DNA"><a href="#DNA" class="headerlink" title="DNA"></a>DNA</h2><p><a href="https://science.howstuffworks.com/life/cellular-microscopic/dna.htm/printable" target="_blank" rel="noopener">How DNA Works | HowStuffWorks</a><br><a href="https://www.chemguide.co.uk/organicprops/aminoacidmenu.html" target="_blank" rel="noopener">amino acids and proteins menu</a><br><a href="https://science-explained.com/theory/dna-rna-and-protein/" target="_blank" rel="noopener">From DNA to RNA to protein, how does it work?</a></p>
<h2 id="Gene-Editing"><a href="#Gene-Editing" class="headerlink" title="Gene Editing"></a>Gene Editing</h2><p><a href="https://www.wikiwand.com/en/CRISPR" target="_blank" rel="noopener">CRISPR - Wikiwand</a><br><a href="https://www.wikiwand.com/en/Indel" target="_blank" rel="noopener">Indel - Wikiwand</a></p>
<p><a href="https://www.broadinstitute.org/what-broad/areas-focus/project-spotlight/questions-and-answers-about-crispr" target="_blank" rel="noopener">Questions and Answers about CRISPR | Broad Institute</a><br><a href="https://gizmodo.com/new-study-finds-unintended-consequences-of-crispr-gene-1827692218/amp" target="_blank" rel="noopener">New Study Finds Unintended Consequences of CRISPR Gene Editing</a><br><a href="https://ghr.nlm.nih.gov/primer/genomicresearch/genomeediting" target="_blank" rel="noopener">What are genome editing and CRISPR-Cas9? - Genetics Home Reference - NIH</a></p>
<p><a href="https://www.wired.com/story/new-science-could-sharpen-crisprs-gene-editing-scalpel/" target="_blank" rel="noopener">Single Base Editing Could Sharpen Crispr&#39;s Genetic Scalpel | WIRED</a><br><a href="https://www.wired.com/2017/06/crispr-mutations/" target="_blank" rel="noopener">Crispr’s Next Big Debate: How Messy Is Too Messy? | WIRED</a><br><a href="https://www.wired.com/2017/05/crispr-snip-away-humanitys-worst-diseases/" target="_blank" rel="noopener">How Crispr Could Snip Away Some of Humanity&#39;s Worst Diseases | WIRED</a></p>
<h2 id="Gender-Determination"><a href="#Gender-Determination" class="headerlink" title="Gender Determination"></a>Gender Determination</h2><p><a href="https://www.wikiwand.com/en/DMRT1" target="_blank" rel="noopener">DMRT1 - Wikiwand</a> if DMRT1 is lost, sperm turns into eggs<br><a href="https://ghr.nlm.nih.gov/gene/DMRT1" target="_blank" rel="noopener">DMRT1 gene - Genetics Home Reference - NIH</a><br><a href="https://www.wikiwand.com/en/Testis-determining_factor" target="_blank" rel="noopener">Testis-determining factor - Wikiwand</a> Sry gene<br><a href="https://ghr.nlm.nih.gov/gene/SRY" target="_blank" rel="noopener">SRY gene - Genetics Home Reference - NIH</a></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5676419/" target="_blank" rel="noopener">Sex determination and maintenance: the role of DMRT1 and FOXL2</a></p>
<p><a href="https://www.wnycstudios.org/shows/radiolab/projects/radiolab-presents-gonads" target="_blank" rel="noopener">Radiolab | Radiolab Presents: Gonads | WNYC Studios</a><br><a href="https://www.wnycstudios.org/story/theprimordialjourney/" target="_blank" rel="noopener">The Primordial Journey | Radiolab | WNYC Studios</a><br><a href="https://www.wnycstudios.org/story/gonads-xy" target="_blank" rel="noopener">X &amp; Y | Radiolab | WNYC Studios</a><br>Stem cells journey their way to gonads, resisting signals to specialize.</p>

	
	</div>
	
  
</div>
	<a type="button" href="/gene-science/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-11-20 </div>
			<div class="article-title"><a href="/web-scraping/" >Web Scraping</a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  <p><a href="https://www.wikiwand.com/en/Web_crawler" target="_blank" rel="noopener">Web crawler - Wikiwand</a><br><a href="https://www.wikiwand.com/en/Web_scraping" target="_blank" rel="noopener">Web scraping - Wikiwand</a></p>
<p><a href="https://github.com/lorien/awesome-web-scraping" target="_blank" rel="noopener">lorien/awesome-web-scraping: List of libraries, tools and APIs for web scraping and data processing.</a><br><a href="https://github.com/BruceDone/awesome-crawler" target="_blank" rel="noopener">BruceDone/awesome-crawler: A collection of awesome web crawler,spider in different languages</a><br><a href="https://blog.scrapinghub.com/" target="_blank" rel="noopener">THE SCRAPINGHUB BLOG</a></p>
<p><a href="https://itsvit.com/blog/big-data-what-is-web-scraping-and-how-to-use-it/" target="_blank" rel="noopener">Big Data: What is Web Scraping and how to use it | IT Svit Blog</a><br><a href="https://itsvit.com/big-data/big-data-scraping-vs-web-data-crawling/" target="_blank" rel="noopener">Big Data Scraping vs Web Data Crawling | IT Svit Blog</a></p>
<p><a href="https://dataflowkit.com/" target="_blank" rel="noopener">Turn Websites into structured data /Dataflow kit</a><br><a href="https://www.diffbot.com/" target="_blank" rel="noopener">Knowledge Graph, AI Web Data Extraction and Crawling | Diffbot</a></p>
<p><a href="https://github.com/s0md3v/Photon" target="_blank" rel="noopener">s0md3v/Photon: Incredibly fast crawler designed for OSINT.</a></p>
<p><a href="https://github.com/cyrus-and/chrome-remote-interface" target="_blank" rel="noopener">cyrus-and/chrome-remote-interface: Chrome Debugging Protocol interface for Node.js</a></p>
<p><a href="https://github.com/postlight/mercury-parser" target="_blank" rel="noopener">postlight/mercury-parser at devawesome</a></p>
<p><a href="https://medium.com/@kensoh/chromeless-chrominator-chromy-navalia-lambdium-ghostjs-autogcd-ef34bcd26907" target="_blank" rel="noopener">Chromeless, Chrominator, Chromy, Navalia, Lambdium, GhostJS, AutoGCD</a></p>
<p><a href="https://chrome.google.com/webstore/detail/selectorgadget/mhjhnkcfbdhnjickkkdbjoemdmbfginb/related" target="_blank" rel="noopener">SelectorGadget - Chrome Web Store</a></p>
<h2 id="Headless-Chrome"><a href="#Headless-Chrome" class="headerlink" title="Headless Chrome"></a>Headless Chrome</h2><p><a href="https://medium.com/@kensoh/chromeless-chrominator-chromy-navalia-lambdium-ghostjs-autogcd-ef34bcd26907" target="_blank" rel="noopener">Chromeless, Chrominator, Chromy, Navalia, Lambdium, GhostJS, AutoGCD</a></p>
<p><a href="https://github.com/graphcool/chromeless" target="_blank" rel="noopener">graphcool/chromeless: 🖥 Chrome automation made simple. Runs locally or headless on AWS Lambda.</a><br><a href="https://github.com/joelgriffith/navalia" target="_blank" rel="noopener">joelgriffith/navalia: A bullet-proof, fast, and reliable headless browser API</a><br><a href="https://github.com/yujiosaka/headless-chrome-crawler" target="_blank" rel="noopener">yujiosaka/headless-chrome-crawler: Distributed crawler powered by Headless Chrome</a></p>
<h3 id="Puppeteer"><a href="#Puppeteer" class="headerlink" title="Puppeteer"></a>Puppeteer</h3><p><a href="https://github.com/GoogleChrome/puppeteer" target="_blank" rel="noopener">GoogleChrome/puppeteer: Headless Chrome Node API</a> !important<br><a href="https://try-puppeteer.appspot.com/" target="_blank" rel="noopener">Try Puppeteer</a><br><a href="https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#class-page" target="_blank" rel="noopener">puppeteer/api.md at master · GoogleChrome/puppeteer</a></p>
<p><a href="https://flaviocopes.com/puppeteer/" target="_blank" rel="noopener">Introduction to Puppeteer</a><br><a href="https://tutorialzine.com/2017/08/automating-google-chrome-with-node-js" target="_blank" rel="noopener">Automating Google Chrome with Node.js - Tutorialzine</a> with Puppeteer<br><a href="https://hackernoon.com/more-than-you-want-to-know-about-headless-chrome-31f6b3b06d82" target="_blank" rel="noopener">More than you want to know about headless Chrome – Hacker Noon</a><br><a href="https://medium.com/@e_mad_ehsan/getting-started-with-puppeteer-and-chrome-headless-for-web-scrapping-6bf5979dee3e" target="_blank" rel="noopener">Getting started with Puppeteer and Chrome Headless for Web Scraping</a><br><a href="https://www.valentinog.com/blog/ui-testing-jest-puppetteer/" target="_blank" rel="noopener">Tutorial: User Interface Testing with Jest and Puppeteer</a><br><a href="https://nemethgergely.com/puppeteer-browser-automation/" target="_blank" rel="noopener">Browser automation revisited - meet Puppeteer</a><br><a href="https://zwischenzugs.com/2017/10/16/puppeteer-headless-chrome-in-a-container/" target="_blank" rel="noopener">Puppeteer – Headless Chrome in a Container – zwischenzugs</a></p>
<p><a href="https://developers.google.com/web/updates/2017/04/headless-chrome" target="_blank" rel="noopener">Getting Started with Headless Chrome  |  Web  |  Google Developers</a><br><a href="https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md" target="_blank" rel="noopener">Headless Chromium</a></p>
<h2 id="Python"><a href="#Python" class="headerlink" title="Python"></a>Python</h2><p><a href="https://github.com/binux/pyspider" target="_blank" rel="noopener">binux/pyspider: A Powerful Spider(Web Crawler) System in Python.</a><br><a href="http://docs.pyspider.org/en/latest/" target="_blank" rel="noopener">Introduction - pyspider</a></p>
<p><a href="https://github.com/MechanicalSoup/MechanicalSoup" target="_blank" rel="noopener">MechanicalSoup/MechanicalSoup: A Python library for automating interaction with websites.</a><br><a href="https://mechanicalsoup.readthedocs.io/en/stable/" target="_blank" rel="noopener">Welcome to MechanicalSoup’s documentation!</a></p>
<p><a href="https://realpython.com/python-web-scraping-practical-introduction/" target="_blank" rel="noopener">Practical Introduction to Web Scraping in Python – Real Python</a><br><a href="https://pythontips.com/2018/06/20/an-intro-to-web-scraping-with-lxml-and-python/" target="_blank" rel="noopener">An Intro to Web Scraping With lxml and Python – Python Tips</a></p>
<p><a href="https://www.youtube.com/playlist?list=PLQVvvaa0QuDfV1MIRBOcqClP6VZXsvyZS" target="_blank" rel="noopener">Web scraping and parsing with Beautiful Soup &amp; Python - YouTube</a></p>
<p><a href="https://newspaper.readthedocs.io/en/latest/" target="_blank" rel="noopener">Newspaper3k: Article scraping &amp; curation — newspaper documentation</a><br><a href="https://github.com/jmcarp/robobrowser" target="_blank" rel="noopener">jmcarp/robobrowser</a><br><a href="https://github.com/michaelhelmick/lassie" target="_blank" rel="noopener">michaelhelmick/lassie: Web Content Retrieval for Humans™</a><br><a href="https://github.com/chineking/cola" target="_blank" rel="noopener">chineking/cola: A high-level distributed crawling framework.</a><br><a href="https://github.com/matiasb/demiurge" target="_blank" rel="noopener">matiasb/demiurge: PyQuery-based scraping micro-framework.</a></p>
<h3 id="Scrapy"><a href="#Scrapy" class="headerlink" title="Scrapy"></a>Scrapy</h3><p><a href="https://scrapy.org/" target="_blank" rel="noopener">Scrapy | A Fast and Powerful Scraping and Web Crawling Framework</a><br><a href="https://doc.scrapy.org/en/latest/" target="_blank" rel="noopener">Scrapy documentation</a><br><a href="https://scrapinghub.com/scrapy-cloud" target="_blank" rel="noopener">Scrapy Cloud</a> Scraping as a Service, with 7 day data retention</p>
<p><a href="https://blog.scrapinghub.com/" target="_blank" rel="noopener">The Scrapinghub Blog – Turn Web Content Into Useful Data</a><br><a href="https://blog.scrapinghub.com/2017/07/07/scraping-the-steam-game-store-with-scrapy/" target="_blank" rel="noopener">Scraping the Steam Game Store with Scrapy – The Scrapinghub Blog</a><br><a href="https://blog.scrapinghub.com/2016/11/24/how-to-build-your-own-price-monitoring-tool/" target="_blank" rel="noopener">How to Build your own Price Monitoring Tool – The Scrapinghub Blog</a></p>
<p><a href="https://github.com/scrapinghub/portia" target="_blank" rel="noopener">scrapinghub/portia: Visual scraping for Scrapy</a><br><a href="https://portia.scrapinghub.com/" target="_blank" rel="noopener">Portia</a> hosted version</p>
<h2 id="JavaScript"><a href="#JavaScript" class="headerlink" title="JavaScript"></a>JavaScript</h2><p><a href="https://blog.bitsrc.io/https-blog-bitsrc-io-how-to-perform-web-scraping-using-node-js-5a96203cb7cb" target="_blank" rel="noopener">How to perform web-scraping using Node.js – Bits and Pieces</a><br><a href="https://blog.bitsrc.io/how-to-perform-web-scraping-using-node-js-part-2-7a365aeedb43" target="_blank" rel="noopener">How to Perform Web-Scraping using Node.js- Part 2 – Bits and Pieces</a></p>
<p><a href="https://github.com/cheeriojs/cheerio" target="_blank" rel="noopener">cheeriojs/cheerio: Fast, flexible, and lean implementation of core jQuery designed specifically for the server.</a></p>
<h2 id="Others"><a href="#Others" class="headerlink" title="Others"></a>Others</h2><p><a href="https://www.devdungeon.com/content/web-scraping-go" target="_blank" rel="noopener">Web Scraping with Go | DevDungeon</a><br><a href="https://github.com/PuerkitoBio/goquery" target="_blank" rel="noopener">PuerkitoBio/goquery: A little like that j-thing, only in Go.</a></p>
<p><a href="https://github.com/segmentio/nightmare" target="_blank" rel="noopener">segmentio/nightmare: A high-level browser automation library.</a> uses Electron (Chromium inside)<br><a href="http://casperjs.org/" target="_blank" rel="noopener">CasperJS, a navigation scripting and testing utility for PhantomJS and SlimerJS</a> Webkit/Gecko</p>
<p><a href="http://teamcapybara.github.io/capybara/" target="_blank" rel="noopener">Capybara</a> <a href="https://github.com/teamcapybara/capybara" target="_blank" rel="noopener">source</a> Ruby, multiple drivers<br><a href="https://blog.codeship.com/capybara-selenium-testing/" target="_blank" rel="noopener">Capybara and Selenium for Testing and Scraping - via @codeship | via @codeship</a></p>

	
	</div>
	
  
</div>
	<a type="button" href="/web-scraping/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-11-15 </div>
			<div class="article-title"><a href="/slayers/" >Slayers</a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  <p><a href="https://www.wikiwand.com/zh/秀逗魔導士" target="_blank" rel="noopener">秀逗魔導士 - Wikiwand</a><br><a href="https://www.wikiwand.com/zh/秀逗魔導士_%28動畫" target="_blank" rel="noopener">秀逗魔導士 (動畫) - Wikiwand</a>)<br><a href="https://www.wikiwand.com/zh/秀逗魔導士角色列表" target="_blank" rel="noopener">秀逗魔導士角色列表 - Wikiwand</a></p>
<p><a href="https://baike.baidu.com/item/秀逗魔导士/7651?fromtitle=秀逗魔导士小说&fromid=9556828" target="_blank" rel="noopener">秀逗魔导士（神坂一著轻小说）_百度百科</a></p>
<p><a href="https://forum.gamer.com.tw/B.php?bsn=40009&subbsn=0" target="_blank" rel="noopener">秀逗魔導士 哈啦板 - 巴哈姆特</a></p>
<p>在本作的世界觀中，有無數個世界同時存在，而目前所知的有四個平行世界運轉著，四界是指：赤之世界、黑之世界、青之世界、白之世界；本作大部分人物所存在的世界為赤之世界。而在這四個世界當中，每個世界都有魔族和神族互相牽制著，這就是世界平衡的狀態。</p>
<p><a href="https://www.wikiwand.com/ja/スレイヤーズの超常的存在" target="_blank" rel="noopener">スレイヤーズの超常的存在 - Wikiwand</a><br><a href="https://baike.baidu.com/item/魔族/20814954" target="_blank" rel="noopener">魔族（《秀逗魔导士》种族）_百度百科</a></p>
<h2 id="四界之上的存在"><a href="#四界之上的存在" class="headerlink" title="四界之上的存在"></a>四界之上的存在</h2><p><a href="https://www.wikiwand.com/zh/%E7%A7%80%E9%80%97%E9%AD%94%E5%B0%8E%E5%A3%AB%E8%A7%92%E8%89%B2%E5%88%97%E8%A1%A8#/.E7.A5.9E.E9.AD.94.E4.B9.8B.E4.B8.8A.E7.9A.84.E5.AD.98.E5.9C.A8" target="_blank" rel="noopener">噩梦之主 Lord of Nightmare</a></p>
<p>重破斬（ギガ·スレイブ）, 其威力大到必須將全身魔力耗盡的地步, 稍一不慎就會毁滅世界<br>在Next 對冥王時，莉娜使出了完全的重破斬把噩梦之主召喚到人間</p>
<p>神滅斬（ラグナ．ブレード）, 莉娜接觸了「異界默示錄（クレアバイブル）」後發展出的咒文</p>
<h2 id="魔族"><a href="#魔族" class="headerlink" title="魔族"></a>魔族</h2><h3 id="紅寶石之眼-赤眼魔王-夏布拉尼古德（（ルビーアイ）シャブラニグドウ）"><a href="#紅寶石之眼-赤眼魔王-夏布拉尼古德（（ルビーアイ）シャブラニグドウ）" class="headerlink" title="紅寶石之眼 赤眼魔王 夏布拉尼古德（（ルビーアイ）シャブラニグドウ）"></a>紅寶石之眼 赤眼魔王 夏布拉尼古德（（ルビーアイ）シャブラニグドウ）</h3><p>龍破斬（ドラグ・スレイブ）、魔王劍（ルビーアイ・ブレード）</p>
<p>雷恩‧麥格納斯進行降魔戰爭, 以龍破斬一擊打倒1600歲的銀雷龍（アーク・ドラゴン）而命名</p>
<h3 id="散播黑暗者-黑暗之星-杜古拉迪古德（闇を撒くもの（ダーク．スター）デュグラディグドゥ）"><a href="#散播黑暗者-黑暗之星-杜古拉迪古德（闇を撒くもの（ダーク．スター）デュグラディグドゥ）" class="headerlink" title="散播黑暗者 黑暗之星 杜古拉迪古德（闇を撒くもの（ダーク．スター）デュグラディグドゥ）"></a>散播黑暗者 黑暗之星 杜古拉迪古德（闇を撒くもの（ダーク．スター）デュグラディグドゥ）</h3><p>散播黑暗者分身的五樣武器</p>
<ul>
<li>烈光之劍（ゴルンノヴァ） (高里所擁有的光之劍, 魔獸沙那法滅亡賽拉格, 被光之戰士打倒, 形成瘴氣之森, 神聖樹福拉菇吸收瘴氣而生)</li>
<li>颶風弓（ガルヴェイラ）</li>
<li>毒牙爪（ネザード）</li>
<li>瞬撃槍（ラグド・メゼギス）</li>
<li>破神槌（ボーディガー）</li>
</ul>
<p>在劇情原創的動畫第三部-TRY中, 黑暗之星將漆黑的龍神 瓦爾菲德吞噬, 變得狂暴化，不分敵我，從此異界的人們陷入長期與黑暗之星對抗的惡夢。</p>
<h3 id="蒼穹之王-混沌之藍（蒼穹の王（カオティック・ブルー））"><a href="#蒼穹之王-混沌之藍（蒼穹の王（カオティック・ブルー））" class="headerlink" title="蒼穹之王 混沌之藍（蒼穹の王（カオティック・ブルー））"></a>蒼穹之王 混沌之藍（蒼穹の王（カオティック・ブルー））</h3><h3 id="白霧-死亡之霧（白霧（デス・フォッグ））"><a href="#白霧-死亡之霧（白霧（デス・フォッグ））" class="headerlink" title="白霧 死亡之霧（白霧（デス・フォッグ））"></a>白霧 死亡之霧（白霧（デス・フォッグ））</h3><h3 id="赤眼魔王五心腹"><a href="#赤眼魔王五心腹" class="headerlink" title="赤眼魔王五心腹"></a>赤眼魔王五心腹</h3><ul>
<li>冥王 菲布里佐（冥王（ヘルマスター）フィブリゾ）<br>冥王幻朧咒（ラグナ・ドライブ）、冥王降魔陣（ラグナ・ブラスト）<br>計畫逼迫莉娜使用重破斬；其目的是要讓混沌之力失去控制，藉以毀滅世界，沒料到重破斬的意義與他的想像不同，反而被金色魔王親手毀滅。<br>在降魔战争中失去了所有部下, 向獸王借用了傑洛士。</li>
<li>魔龍王 加布（魔竜王（カオスドラゴン）ガーヴ）<br>魔龍烈火咆（ガーヴ・フレア）<br>冥王佈下陷阱，使魔龍王與莉娜戰鬥，當莉娜出完全版神滅斬消耗加布的力量後，冥王正式現身輕鬆消滅已經精疲力盡的加布。</li>
<li>獸王 潔拉絲（獣王（グレーター・ビースト）ゼラス＝メタリオム）<br>獸王牙操彈（ゼラス・ブリッド）</li>
<li>海王 達爾菲（海王（デイーブシー）ダルフイン）<br>海王滅殺斬（ダルフ・ゾーク）、海王槍破擊（ダルフ・ストラッシュ）</li>
<li>霸王 古勞雪拉（霸王（ダイナスト）グラウシエラ）<br>霸王雷擊陣（ダイナスト・ブラス）、霸王冰河烈（ダイナスト・ブレス）</li>
</ul>
<h3 id="高位魔族"><a href="#高位魔族" class="headerlink" title="高位魔族"></a>高位魔族</h3><ul>
<li>獸神官 傑洛士（獣神官（プリースト）ゼロス）<br>獸王避免力量分散，只設立了神官而沒有將軍，傑洛士實力僅在五王之下。</li>
</ul>
<h2 id="神族"><a href="#神族" class="headerlink" title="神族"></a>神族</h2><h3 id="赤龍神-史菲德（赤の竜神（スイーフイード））"><a href="#赤龍神-史菲德（赤の竜神（スイーフイード））" class="headerlink" title="赤龍神 史菲德（赤の竜神（スイーフイード））"></a>赤龍神 史菲德（赤の竜神（スイーフイード））</h3><p>將赤眼魔王分成七份封入人體後，留下四個分身之後毀滅</p>
<ul>
<li>火龍王 瓦拉巴薩德（火竜王（フレアロード）ヴラバザード）</li>
<li>水龍王 拉格拉提亞（水竜王（アクアロード）ラグラテイア）<br>千年前與赤眼魔王碎片, 五賢者之一的雷恩‧麥格納斯進行降魔戰爭,把魔龍王 加布封印在人類的身體</li>
<li>空龍王 巴爾溫（空竜王（エアロード）バールウィン）</li>
<li>地龍王 朗戈特（地竜王（アースロード）ランゴート）</li>
</ul>
<h3 id="漆黑的龍神-夜之龍-瓦爾菲德（漆黒の竜神（ナイト・ドラゴン）ヴォルフィード）"><a href="#漆黑的龍神-夜之龍-瓦爾菲德（漆黒の竜神（ナイト・ドラゴン）ヴォルフィード）" class="headerlink" title="漆黑的龍神 夜之龍 瓦爾菲德（漆黒の竜神（ナイト・ドラゴン）ヴォルフィード）"></a>漆黑的龍神 夜之龍 瓦爾菲德（漆黒の竜神（ナイト・ドラゴン）ヴォルフィード）</h3><h3 id="白龍神"><a href="#白龍神" class="headerlink" title="白龍神"></a>白龍神</h3><h3 id="青龍神"><a href="#青龍神" class="headerlink" title="青龍神"></a>青龍神</h3><h2 id="龍族"><a href="#龍族" class="headerlink" title="龍族"></a>龍族</h2><h3 id="米爾蓋茲亞-（ミルガズィア）"><a href="#米爾蓋茲亞-（ミルガズィア）" class="headerlink" title="米爾蓋茲亞 （ミルガズィア）"></a>米爾蓋茲亞 （ミルガズィア）</h3><p>黃金龍族（Golden Dragon ゴールデン·ドラゴン）的长老</p>
<h3 id="菲利亞·烏爾·科普特（フィリア＝ウル＝コプト）"><a href="#菲利亞·烏爾·科普特（フィリア＝ウル＝コプト）" class="headerlink" title="菲利亞·烏爾·科普特（フィリア＝ウル＝コプト）"></a>菲利亞·烏爾·科普特（フィリア＝ウル＝コプト）</h3><p>黃金龍族的女巫</p>
<h3 id="瓦爾加布（ヴァルガーヴ）"><a href="#瓦爾加布（ヴァルガーヴ）" class="headerlink" title="瓦爾加布（ヴァルガーヴ）"></a>瓦爾加布（ヴァルガーヴ）</h3><p>古代龍族（Ancient Dragon）的僅存後裔, 被魔龍王加布所救<br>得知加布死訊後, 立誓要為加布向莉娜一行人報仇<br>與黑暗之星同化, 被淨化回到幼龍的型態並成為菲利亞的養子</p>
<hr>
<p><a href="https://www.wikiwand.com/zh/秀逗魔導士#/.E9.AD.94.E6.B3.95" target="_blank" rel="noopener">秀逗魔導士 - 魔法 - Wikiwand</a></p>
<h2 id="降魔战争"><a href="#降魔战争" class="headerlink" title="降魔战争"></a>降魔战争</h2><p><a href="https://baike.baidu.com/item/降魔战争" target="_blank" rel="noopener">降魔战争_百度百科</a></p>
<p><a href="https://forum.gamer.com.tw/G2.php?bsn=40009&sn=115" target="_blank" rel="noopener">降魔戰爭的詳細歷史文獻 @秀逗魔導士 精華區 - 巴哈姆特</a></p>

	
	</div>
	
  
</div>
	<a type="button" href="/slayers/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-11-03 </div>
			<div class="article-title"><a href="/docker-nvidia/" >Nvidia GPU in Docker</a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  <p><a href="https://developer.nvidia.com/nvidia-container-runtime" target="_blank" rel="noopener">NVIDIA Container Runtime | NVIDIA Developer</a><br><a href="https://devblogs.nvidia.com/gpu-containers-runtime/" target="_blank" rel="noopener">Enabling GPUs in the Container Runtime Ecosystem | NVIDIA Developer Blog</a></p>
<p><a href="https://github.com/NVIDIA/nvidia-docker" target="_blank" rel="noopener">NVIDIA/nvidia-docker: Build and run Docker containers leveraging NVIDIA GPUs</a><br><a href="https://github.com/NVIDIA/nvidia-docker/releases" target="_blank" rel="noopener">Releases · NVIDIA/nvidia-docker</a> <a href="https://github.com/NVIDIA/nvidia-docker/wiki/Motivation" target="_blank" rel="noopener">Motivation · NVIDIA/nvidia-docker Wiki</a><br>2.0 uses <code>nvidia-container-runtime</code> instead of <code>runC</code>.</p>
<p><a href="https://nvidia.github.io/libnvidia-container/" target="_blank" rel="noopener">Repository configuration | libnvidia-container</a><br><a href="https://github.com/nvidia/nvidia-container-runtime#installation" target="_blank" rel="noopener">NVIDIA/nvidia-container-runtime: NVIDIA container runtime</a> a modified runC that will invoke <code>nvidia-container-cli</code> from project <code>libnvidia-container</code> when container starts.<br><a href="https://github.com/NVIDIA/libnvidia-container" target="_blank" rel="noopener">NVIDIA/libnvidia-container: NVIDIA container runtime library</a></p>
<h1 id="Installation"><a href="#Installation" class="headerlink" title="Installation"></a>Installation</h1><p><a href="https://github.com/NVIDIA/nvidia-docker/wiki/Installation-%28version-2.0%29" target="_blank" rel="noopener">Installation (version 2.0) · NVIDIA/nvidia-docker Wiki</a></p>
<h2 id="Docker"><a href="#Docker" class="headerlink" title="Docker"></a>Docker</h2><ul>
<li>Installation Ubuntu 16.04 Server (16.04 with upgrade)</li>
<li>Install Docker CE (17.06+) with.deb<br><a href="https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#install-from-a-package" target="_blank" rel="noopener">https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#install-from-a-package</a></li>
</ul>
<figure class="highlight sh"><table><tr><td class="code"><pre><span class="line">sudo apt install libltdl7</span><br><span class="line">sudo dpkg -i docker-ce_17.06.2-ce-0-ubuntu_amd64.deb</span><br><span class="line">sudo systemctl <span class="built_in">enable</span> docker &amp;&amp; sudo systemctl start docker</span><br><span class="line">sudo usermod -a -G docker <span class="variable">$&#123;USER&#125;</span></span><br></pre></td></tr></table></figure>

<h2 id="NVIDIA-driver"><a href="#NVIDIA-driver" class="headerlink" title="NVIDIA driver"></a>NVIDIA driver</h2><p><a href="https://github.com/NVIDIA/nvidia-docker/wiki/Frequently-Asked-Questions#how-do-i-install-the-nvidia-driver" target="_blank" rel="noopener">How do I install the NVIDIA driver? · NVIDIA/nvidia-docker Wiki</a><br><a href="http://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#ubuntu-installation" target="_blank" rel="noopener">Installation Guide Linux :: CUDA Toolkit Documentation</a><br><a href="https://www.linkedin.com/pulse/installing-nvidia-cuda-80-ubuntu-1604-linux-gpu-new-victor" target="_blank" rel="noopener">Installing Nvidia CUDA 8.0 on Ubuntu 16.04 for Linux GPU Computing (New Troubleshooting Guide)</a></p>
<p>install NVIDIA driver (384.90+) and CUDA</p>
<figure class="highlight sh"><table><tr><td class="code"><pre><span class="line">sudo apt-get install nvidia-384 nvidia-cuda-toolkit</span><br><span class="line"></span><br><span class="line"><span class="comment"># verify installation</span></span><br><span class="line">lsmod | grep nvidia</span><br><span class="line">ldconfig -p | grep -E <span class="string">'nvidia|cuda'</span></span><br><span class="line"></span><br><span class="line">nvidia-smi</span><br><span class="line">nvidia-modprobe</span><br><span class="line">nvcc --version</span><br></pre></td></tr></table></figure>

<p><a href="https://comzyh.com/blog/archives/967/" target="_blank" rel="noopener">解决 Driver/library version mismatch | Comzyh 的博客</a><br>Also fixed by a reboot<br><a href="https://launchpad.net/~graphics-drivers/+archive/ubuntu/ppa" target="_blank" rel="noopener">Proprietary GPU Drivers : “Graphics Drivers” team</a><br><a href="https://blog.csdn.net/xianglao1935/article/details/80512345" target="_blank" rel="noopener">https://blog.csdn.net/xianglao1935/article/details/80512345</a></p>
<h2 id="nvidia-docker2"><a href="#nvidia-docker2" class="headerlink" title="nvidia-docker2"></a><code>nvidia-docker2</code></h2><blockquote>
<p>Note: installing this overwrites <code>/etc/docker/daemon.json</code></p>
</blockquote>
<p>Set default runtime of docker daemon with <code>--default-runtime=nvidia</code>.<br><a href="https://github.com/nvidia/nvidia-container-runtime#environment-variables-oci-spec" target="_blank" rel="noopener">Environment variables (OCI spec)</a></p>
<blockquote>
<p>Use <code>nvidia-docker2</code> matching the Docker version installed</p>
</blockquote>
<figure class="highlight sh"><table><tr><td class="code"><pre><span class="line">curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | \</span><br><span class="line">  sudo apt-key add -</span><br><span class="line">curl -s -L https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64/nvidia-docker.list | \</span><br><span class="line">  sudo tee /etc/apt/sources.list.d/nvidia-docker.list</span><br><span class="line">sudo apt-get update</span><br><span class="line">sudo apt-get install nvidia-docker2  <span class="comment"># latest</span></span><br><span class="line"></span><br><span class="line"><span class="comment"># MUST match the Docker version installed</span></span><br><span class="line">apt show nvidia-docker2 -a | grep Version</span><br><span class="line">apt show nvidia-container-runtime -a | grep Version</span><br><span class="line"><span class="comment"># for Docker 17.06.2</span></span><br><span class="line">sudo apt-get install \</span><br><span class="line">  nvidia-docker2=2.0.3+docker17.06.2-1 \</span><br><span class="line">  nvidia-container-runtime=2.0.0+docker17.06.2-1</span><br><span class="line"><span class="comment"># for Docker 17.09</span></span><br><span class="line">sudo apt-get install \</span><br><span class="line">  nvidia-docker2=2.0.3+docker17.09.0-1 \</span><br><span class="line">  nvidia-container-runtime=2.0.0+docker17.09.0-1</span><br><span class="line">sudo pkill -SIGHUP dockerd</span><br></pre></td></tr></table></figure>

<h2 id="Changing-default-runtime"><a href="#Changing-default-runtime" class="headerlink" title="Changing default runtime"></a>Changing default runtime</h2><p>Use <code>/usr/bin/nvidia-container-runtime</code> in <code>/etc/docker/daemon.json</code>:</p>
<figure class="highlight json"><table><tr><td class="code"><pre><span class="line">&#123;</span><br><span class="line">  <span class="attr">"default-runtime"</span>: <span class="string">"nvidia"</span>,</span><br><span class="line">  <span class="attr">"runtimes"</span>: &#123;</span><br><span class="line">    <span class="attr">"nvidia"</span>: &#123;</span><br><span class="line">      <span class="attr">"path"</span>: <span class="string">"/usr/bin/nvidia-container-runtime"</span>,</span><br><span class="line">      <span class="attr">"runtimeArgs"</span>: []</span><br><span class="line">    &#125;</span><br><span class="line">  &#125;</span><br><span class="line">&#125;</span><br></pre></td></tr></table></figure>

<figure class="highlight sh"><table><tr><td class="code"><pre><span class="line"><span class="comment"># now use</span></span><br><span class="line">docker run --rm nvidia/cuda:8.0-runtime-ubuntu16.04 nvidia-smi</span><br><span class="line"><span class="comment"># instead of</span></span><br><span class="line">docker run --runtime=nvidia --rm nvidia/cuda:8.0-runtime-ubuntu16.04 nvidia-smi</span><br><span class="line"></span><br><span class="line"><span class="comment"># swarm service also work</span></span><br><span class="line">docker swarm init</span><br><span class="line">docker service create --name cuda-service --constraint node.labels.gpu==<span class="literal">true</span> nvidia/cuda:<span class="built_in">test</span>-service</span><br></pre></td></tr></table></figure>

<h2 id="cuda-Base-Image"><a href="#cuda-Base-Image" class="headerlink" title="cuda Base Image"></a>cuda Base Image</h2><p><a href="https://hub.docker.com/r/nvidia/cuda/" target="_blank" rel="noopener">nvidia/cuda - Docker Hub</a><br><a href="https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/8.0/runtime/Dockerfile" target="_blank" rel="noopener">8.0/runtime/Dockerfile · ubuntu16.04 · nvidia / cuda · GitLab</a><br><a href="https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/8.0/devel/Dockerfile" target="_blank" rel="noopener">8.0/devel/Dockerfile · ubuntu16.04 · nvidia / cuda · GitLab</a><br><a href="https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/8.0/runtime/cudnn8/Dockerfile" target="_blank" rel="noopener">8.0/runtime/cudnn8/Dockerfile · ubuntu16.04 · nvidia / cuda · GitLab</a></p>
<h2 id="Kubernetes"><a href="#Kubernetes" class="headerlink" title="Kubernetes"></a>Kubernetes</h2><p><a href="https://developer.nvidia.com/kubernetes-gpu" target="_blank" rel="noopener">NVIDIA Container Runtime and Orchestrators | NVIDIA Developer</a><br><a href="https://docs.nvidia.com/datacenter/kubernetes/kubernetes-install-guide/index.html" target="_blank" rel="noopener">Kubernetes on NVIDIA GPUs Installation Guide :: Data Center Documentation</a></p>
<p><a href="https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/" target="_blank" rel="noopener">Schedule GPUs - Kubernetes</a><br><a href="https://ubuntu.com/blog/using-gpgpus-with-kubernetes" target="_blank" rel="noopener">Using GPGPUs with Kubernetes | Ubuntu</a><br><a href="https://www.infoq.cn/article/ypP*1sbAuBAD1KL1qB4K" target="_blank" rel="noopener">基于 Kubernetes 的 GPU 类型调度实现</a></p>

	
	</div>
	
  
</div>
	<a type="button" href="/docker-nvidia/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-10-31 </div>
			<div class="article-title"><a href="/math/" >Mathematics</a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  <p><a href="https://www.coursera.org/specializations/discrete-mathematics" target="_blank" rel="noopener">Introduction to Discrete Mathematics for Computer Science | Coursera</a></p>
<p><a href="https://www.youtube.com/channel/UCq0EGvLTyy-LLT1oUSO_0FQ" target="_blank" rel="noopener">Eddie Woo - YouTube - YouTube</a></p>
<p><a href="https://mathwithbaddrawings.com/" target="_blank" rel="noopener">Math with Bad Drawings – Lover of math. Bad at drawing.</a></p>
<p><a href="https://dev.to/trekhleb/permutationscombinations-algorithms-cheat-sheets-3job" target="_blank" rel="noopener">Permutations/Combinations Algorithms Cheat Sheets - DEV Community 👩‍💻👨‍💻</a></p>
<p><a href="http://www.coolmath.com/" target="_blank" rel="noopener">Cool Math - free online cool math lessons, cool math games &amp; apps, fun math activities, pre-algebra, algebra, precalculus</a><br><a href="https://www.coolmath4kids.com" target="_blank" rel="noopener">CoolMath4Kids</a></p>
<p><a href="https://www.c82.net/euclid/" target="_blank" rel="noopener">Byrne’s Euclid</a></p>
<p><a href="https://www.nature.com/articles/d41586-019-00083-3" target="_blank" rel="noopener">Machine learning leads mathematicians to unsolvable problem</a></p>
<p><a href="https://www.youtube.com/channel/UCLROLAN8kmU7tGQDs6KH-bQ" target="_blank" rel="noopener">妈咪说 MommyTalk - YouTube</a></p>
<p><a href="https://www.quora.com/Whats-the-easiest-way-to-understand-G%C3%B6dels-Incompleteness-Theorems-Are-there-statements-that-have-truth-values-which-cannot-be-determined-except-meta-mathematically" target="_blank" rel="noopener">(1) What&#39;s the easiest way to understand Gödel&#39;s Incompleteness Theorems? Are there statements that have truth values which cannot be determined except meta-mathematically? - Quora</a></p>
<p><a href="https://www.youtube.com/playlist?list=PLn8PRpmsu08ojy02wi4QLVzELM545Xw3p" target="_blank" rel="noopener">Understanding Wavelets - YouTube</a></p>
<h2 id="Linear-Algebra"><a href="#Linear-Algebra" class="headerlink" title="Linear Algebra"></a>Linear Algebra</h2><p><a href="https://www.wikiwand.com/en/Linear_algebra" target="_blank" rel="noopener">Linear algebra - Wikiwand</a><br><a href="https://www.wikiwand.com/en/Eigenvalues_and_eigenvectors" target="_blank" rel="noopener">Eigenvalues and eigenvectors - Wikiwand</a></p>
<p><a href="https://www.youtube.com/playlist?list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab" target="_blank" rel="noopener">Essence of linear algebra - YouTube</a></p>
<h2 id="Group-Theory"><a href="#Group-Theory" class="headerlink" title="Group Theory"></a>Group Theory</h2><p><a href="https://www.wikiwand.com/en/Group_%28mathematics%29" target="_blank" rel="noopener">Group (mathematics) - Wikiwand</a> Set with binary operation defined<br><a href="https://www.wikiwand.com/en/Ring_%28mathematics%29" target="_blank" rel="noopener">Ring (mathematics) - Wikiwand</a> Group closed in addition, subtraction and multiplication<br><a href="https://www.wikiwand.com/en/Field_%28mathematics%29" target="_blank" rel="noopener">Field (mathematics) - Wikiwand</a> Ring closed also in division</p>
<p><a href="https://www.wikiwand.com/en/Galois_theory" target="_blank" rel="noopener">Galois theory - Wikiwand</a><br><a href="https://www.youtube.com/playlist?list=PLYtoePJQbGmiq3lqkJcxSKW0eW5objmjQ" target="_blank" rel="noopener">从五次方程到伽罗瓦理论 - YouTube</a></p>
<h2 id="Calculus"><a href="#Calculus" class="headerlink" title="Calculus"></a>Calculus</h2><p><a href="https://www.youtube.com/playlist?list=PLZHQObOWTQDMsr9K-rj53DwVRMYO3t5Yr" target="_blank" rel="noopener">Essence of calculus - YouTube</a></p>
<h2 id="Pi"><a href="#Pi" class="headerlink" title="Pi"></a>Pi</h2><p><a href="https://www.youtube.com/playlist?list=PLZHQObOWTQDMVQcT3414TcPMeEYf_VtPM" target="_blank" rel="noopener">Why pi? - YouTube</a></p>
<h2 id="Statistics"><a href="#Statistics" class="headerlink" title="Statistics"></a>Statistics</h2><p><a href="http://simplestatistics.org/" target="_blank" rel="noopener">Simple Statistics</a></p>
<p><a href="https://www.youtube.com/watch?v=fl9V0U2SGeI" target="_blank" rel="noopener">Three Statistical Tests Every Game Developer Should Know - YouTube</a></p>
<p><a href="https://greenteapress.com/wp/think-stats-2e/" target="_blank" rel="noopener">Think Stats 2e – Green Tea Press</a></p>
<h2 id="Bayes-Theorem"><a href="#Bayes-Theorem" class="headerlink" title="Bayes Theorem"></a>Bayes Theorem</h2><p><a href="http://greenteapress.com/wp/think-bayes/" target="_blank" rel="noopener">Think Bayes – Green Tea Press</a></p>
<p><a href="https://www.youtube.com/watch?v=5NMxiOGL39M" target="_blank" rel="noopener">How Bayes Theorem works - YouTube</a><br><a href="https://www.youtube.com/watch?v=BrK7X_XlGB8" target="_blank" rel="noopener">A visual guide to Bayesian thinking - YouTube</a></p>

	
	</div>
	
  
</div>
	<a type="button" href="/math/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-10-31 </div>
			<div class="article-title"><a href="/sequence-and-series/" >Sequence and Series</a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  <h2 id="Sequence-and-Series"><a href="#Sequence-and-Series" class="headerlink" title="Sequence and Series"></a>Sequence and Series</h2><p><a href="https://www.youtube.com/watch?v=yudhkUUzAgY" target="_blank" rel="noopener">Grandi&#39;s Series: 1-1+1-1+1... - YouTube</a></p>
<p>A sequence is an ordered list of infinite expressions.<br>A series a sum of infinite expressions.</p>
<p>When calculating the sum of an series, we look at the sequence of partial sums, defined to be the sum up to the n-th terms. If the limit of the partial sums exists and is bound, we assign the limit as the sum of the series. </p>
<p>We can add, subtract terms by terms of series&#39; by adding and subtracting their values.</p>
<p>Alternatively you can tweak the definition of series, say looking at the sequence of the averages of partial sums, if the limit exists (as in Grandi&#39;s Series), it is said to be Cesaro convergent. This trick can be applied repeatedly, look at the sequence of averages of the averages of partial sums. Then some of the previously unbound series are now Cesaro convergent so values can be assigned to it.</p>
<p>When the limit is unbound, the sum of the series is undefined. Applying Riemann Zeta function for s &lt;= 1 with analytic continuation is actually redefining the function for s &lt;= 1. We <em>do not have to</em> make the redefinition to be symmetrical to the s &gt; 1 portion (this is chosen to make the redefinition derivable (analytic)), but if we do, we will <em>assign</em> -1/2 to zeta(-1) and such. It&#39;s not actually taking the limit, it&#39;s just a way to assign value to the symbol zeta(-1).<br>Like we (the human race) did to extend rational numbers to real numbers, then to extend real numbers to complex numbers, all of these series will make sense if we relax the axioms.</p>
<h2 id="Riemann-Hypothesis"><a href="#Riemann-Hypothesis" class="headerlink" title="Riemann Hypothesis"></a>Riemann Hypothesis</h2><p>ζ(s) = 1/1^s+1/2^s+...</p>
<p>s &gt; 1 originally, Riemann ζ function analytically extended the function to complex plane and dropped this restriction</p>
<p>This sum is observed in many physical fields.<br><a href="https://www.youtube.com/watch?v=7AOCCVumSmQ" target="_blank" rel="noopener">自然数之和等于-1/12有何含义？隐藏在卡西米尔效应中的无穷大 - YouTube</a><br><a href="https://www.youtube.com/watch?v=0Oazb7IWzbA" target="_blank" rel="noopener">Why -1/12 is a gold nugget - YouTube</a><br><a href="https://www.youtube.com/watch?v=YuIIjLr6vUA" target="_blank" rel="noopener">Numberphile v. Math: the truth about 1+2+3+...=-1/12 - YouTube</a><br><a href="https://www.youtube.com/watch?v=BpfY8m2VLtc" target="_blank" rel="noopener">Response to Numberphile&#39;s ASTOUNDING 1+2+3+4+... = minus 1/12 (sum of natural numbers to infinity) - YouTube</a><br><a href="https://www.youtube.com/watch?v=jcKRGpMiVTw" target="_blank" rel="noopener">Ramanujan: Making sense of 1+2+3+... = -1/12 and Co. - YouTube</a><br><a href="https://www.youtube.com/watch?v=P913qwtXihk" target="_blank" rel="noopener">The sum of all counting numbers equals WHAT? - YouTube</a></p>
<p><a href="https://www.youtube.com/watch?v=td-O4CTO2XI" target="_blank" rel="noopener">素数（二）黎曼猜想和素数有啥关系？欧拉发现黎曼猜想的金钥匙！ - YouTube</a><br><a href="https://www.youtube.com/watch?v=FHHbG6UFHlc" target="_blank" rel="noopener">素数（三）黎曼猜想是什么？它被证明了吗？十分钟看懂黎曼猜想与解析延拓 - YouTube</a><br><a href="https://www.youtube.com/watch?v=yxpIzf-kxm0" target="_blank" rel="noopener">素数（四）黎曼猜想中的非平凡零点和素数的分布规律有什么关系？ - YouTube</a></p>
<p><a href="https://www.youtube.com/watch?v=T93SayXhw2w" target="_blank" rel="noopener">1+2+3+4+...=-1/12？李永乐老师讲黎曼猜想（1） - YouTube</a><br><a href="https://www.youtube.com/watch?v=4vbcC4TcMGc" target="_blank" rel="noopener">质数有多重要？数学家欧拉和高斯是如何研究质数的 ？李永乐老师讲黎曼猜想（2） - YouTube</a><br><a href="https://www.youtube.com/watch?v=NeoDdnSlRjk" target="_blank" rel="noopener">悬赏100万美元的“黎曼猜想”有多难？李永乐老师讲什么是黎曼猜想（3） - YouTube</a></p>
<p><a href="https://www.youtube.com/watch?v=sD0NjbwqlYw" target="_blank" rel="noopener">Visualizing the Riemann zeta function and analytic continuation - YouTube</a></p>
<p><a href="https://www.youtube.com/watch?v=-EtHF5ND3_s" target="_blank" rel="noopener">Riemann&#39;s paradox: pi = infinity minus infinity - YouTube</a> Riemann Rearrangement Theorem</p>

	
	</div>
	
  
</div>
	<a type="button" href="/sequence-and-series/#more" class="btn btn-default more">Read More</a>
</div>

		
			
	
	<!-- display as entry -->	
		<h3 class="title">
			<div class="date"> 2017-08-18 </div>
			<div class="article-title"><a href="/blockchain/" >Blockchain</a></div>						
		</h3>
	


			<div class="entry">
  <div class="row">
  
	<div class="col-md-12">
	  
	
	  <p><a href="https://medium.com/@cdixon/crypto-tokens-a-breakthrough-in-open-network-design-e600975be2ef" target="_blank" rel="noopener">Crypto Tokens: A Breakthrough in Open Network Design</a></p>
<p><a href="https://www.youtube.com/watch?v=qOVAbKKSH10" target="_blank" rel="noopener">Blockchain Technology Explained (2 Hour Course) - YouTube</a></p>
<p><a href="https://www.computerworld.com/article/3236480/emerging-technology/the-top-5-problems-with-blockchain.html" target="_blank" rel="noopener">The top 5 problems with blockchain | Computerworld</a><br><a href="https://www.computerworld.com/article/3269750/blockchain/will-blockchain-run-afoul-of-gdpr-yes-and-no.html" target="_blank" rel="noopener">Will blockchain run afoul of GDPR? (Yes and no) | Computerworld</a></p>
<p><a href="https://softwareengineeringdaily.com/category/blockchain/" target="_blank" rel="noopener">Blockchain | Software Engineering Daily</a><br><a href="https://softwareengineeringdaily.com/2017/10/12/blockchain-building-with-daniel-van-flymen/" target="_blank" rel="noopener">Blockchain Building with Daniel van Flymen | Software Engineering Daily</a> !important, with Python<br><a href="https://softwareengineeringdaily.com/2015/08/11/blockchains-with-melanie-swan/" target="_blank" rel="noopener">Blockchains with Melanie Swan | Software Engineering Daily</a></p>
<p><a href="https://www.extremetech.com/extreme/262229-bitcoin-blockchain-icos-explained" target="_blank" rel="noopener">Bitcoin, Blockchain, and ICOs: What You Need to Know - ExtremeTech</a></p>
<p><a href="https://github.com/coinpride/CryptoList" target="_blank" rel="noopener">coinpride/CryptoList: Curated collection of blockchain &amp; cryptocurrency resources.</a></p>
<h2 id="Cryptocurrency"><a href="#Cryptocurrency" class="headerlink" title="Cryptocurrency"></a>Cryptocurrency</h2><p><a href="https://hackernoon.com/state-of-cryptocurrencies-summer-2018-932016549375" target="_blank" rel="noopener">State of Cryptocurrencies: Summer 2018 – Hacker Noon</a></p>
<p><a href="http://bitcoinfoundation.org/" target="_blank" rel="noopener">Bitcoin Foundation | Supporting Education, Adoption and Development in Bitcoin</a></p>
<p><a href="https://www.khanacademy.org/economics-finance-domain/core-finance/money-and-banking/bitcoin/v/bitcoin-what-is-it" target="_blank" rel="noopener">Bitcoin: What is it? | Bitcoin | Khan Academy</a><br><a href="https://hackernoon.com/the-ultimate-guide-to-understanding-blockchain-and-cryptocurrencies-f37cf4c0043" target="_blank" rel="noopener">The Ultimate Guide To Understanding The Basics of Blockchain and Cryptocurrencies</a></p>
<p><a href="https://softwareengineeringdaily.com/2017/10/28/cryptoeconomics-with-vlad-zamfir/" target="_blank" rel="noopener">Cryptoeconomics with Vlad Zamfir | Software Engineering Daily</a></p>
<p><a href="https://github.com/Xel/Blockchain-stuff" target="_blank" rel="noopener">Xel/Blockchain-stuff: Blockchain and Crytocurrency Resources</a><br><a href="https://github.com/kennethreitz/awesome-coins" target="_blank" rel="noopener">kennethreitz/awesome-coins: ₿ A guide (for humans!) to cryto-currencies and their algos.</a></p>
<h2 id="Smart-Contracts"><a href="#Smart-Contracts" class="headerlink" title="Smart Contracts"></a>Smart Contracts</h2><p><a href="https://bitsonblocks.net/2016/02/01/a-gentle-introduction-to-smart-contracts/" target="_blank" rel="noopener">A gentle introduction to smart contracts | Bits on blocks</a></p>
<p><a href="https://softwareengineeringdaily.com/2017/10/20/4085/" target="_blank" rel="noopener">Smart Contract Security with Emin Gün Sirer | Software Engineering Daily</a><br><a href="https://softwareengineeringdaily.com/2017/10/11/ethereum-platform-with-preethi-kasireddy/" target="_blank" rel="noopener">Ethereum Platform with Preethi Kasireddy | Software Engineering Daily</a></p>
<h2 id="Asset-Management"><a href="#Asset-Management" class="headerlink" title="Asset Management"></a>Asset Management</h2><p><a href="https://forum.forgerock.com/2016/06/blockchain-identity-access-request-managmement/" target="_blank" rel="noopener">Blockchain For Identity: Access Request Managmement - ForgeRock Forum and Blog</a></p>
<p><a href="https://www.wikiwand.com/en/OpenAM" target="_blank" rel="noopener">OpenAM - Wikiwand</a><br><a href="http://www.theidentitycookbook.com/2016/11/using-openam-as-trusted-file.html" target="_blank" rel="noopener">The Identity Cookbook: Using OpenAM as a Trusted File Authorization Engine</a></p>

	
	</div>
	
  
</div>
	<a type="button" href="/blockchain/#more" class="btn btn-default more">Read More</a>
</div>

		

		</div>

		<!-- pagination -->
		<div>
  		<center>
		<div class="pagination">
<ul class="pagination">
	 
		
    	<li class="prev"><a href="/page/4/" class="alignleft prev"><i class="fa fa-arrow-circle-o-left"></i> Prev</a></li>
  		

        <li><a href="/"><i class="fa fa-home"></i>Home</a></li>

		
		   <li class="next"> <a href="/page/6/" class="alignright next">Next<i class="fa fa-arrow-circle-o-right"></i></a> </li>          
        
	
</ul>
</div>

  		</center>
		</div>

		
		
	</div> <!-- col-md-9 -->

	
		<div class="col-md-3">
	<div id="sidebar">
	
			<div class="form-group has-success has-feedback">
  <form action="//google.com/search" method="get" accept-charset="utf-8" >
    <input type="search" name="q" results="0" placeholder="Search" class="form-control">
    <input type="hidden" name="q" value="site:leesei.github.io">	
  </form>
</div>

		
			
	<div class="widget">
		<h4>Categories</h4>
		<ul class="tag_box inline list-unstyled">
		
			<li><a href="/categories/android/">android<span>4</span></a></li>
		
			<li><a href="/categories/anime/">anime<span>2</span></a></li>
		
			<li><a href="/categories/app/">app<span>50</span></a></li>
		
			<li><a href="/categories/big-data/">big-data<span>2</span></a></li>
		
			<li><a href="/categories/business/">business<span>1</span></a></li>
		
			<li><a href="/categories/comp/">comp<span>10</span></a></li>
		
			<li><a href="/categories/comp-hardware/">comp.hardware<span>24</span></a></li>
		
			<li><a href="/categories/comp-hw/">comp.hw<span>1</span></a></li>
		
			<li><a href="/categories/comp-lang/">comp.lang<span>48</span></a></li>
		
			<li><a href="/categories/compd/">compd<span>1</span></a></li>
		
			<li><a href="/categories/container/">container<span>11</span></a></li>
		
			<li><a href="/categories/trivia/transmission/drivetrain/">drivetrain<span>1</span></a></li>
		
			<li><a href="/categories/ebook/">ebook<span>2</span></a></li>
		
			<li><a href="/categories/games/">games<span>15</span></a></li>
		
			<li><a href="/categories/history/">history<span>12</span></a></li>
		
			<li><a href="/categories/home-improvement/">home-improvement<span>1</span></a></li>
		
			<li><a href="/categories/law/">law<span>4</span></a></li>
		
			<li><a href="/categories/learning/">learning<span>1</span></a></li>
		
			<li><a href="/categories/linux/">linux<span>26</span></a></li>
		
			<li><a href="/categories/math/">math<span>4</span></a></li>
		
			<li><a href="/categories/anime/movie/">movie<span>1</span></a></li>
		
			<li><a href="/categories/philosophy/">philosophy<span>7</span></a></li>
		
			<li><a href="/categories/law/politics/">politics<span>1</span></a></li>
		
			<li><a href="/categories/standard/">standard<span>1</span></a></li>
		
			<li><a href="/categories/toys/">toys<span>2</span></a></li>
		
			<li><a href="/categories/trivia/transmission/">transmission<span>1</span></a></li>
		
			<li><a href="/categories/trivia/">trivia<span>112</span></a></li>
		
			<li><a href="/categories/uncategorized/">uncategorized<span>7</span></a></li>
		
			<li><a href="/categories/web/">web<span>89</span></a></li>
		
		</ul>
	</div>

		
			
	<div class="widget">
		<h4>Tag Cloud</h4>
		<ul class="tag_box inline list-unstyled">		
		
			<li><a href="/tags/network/">network<span>1</span></a></li>
		
			<li><a href="/tags/bazel/">bazel<span>1</span></a></li>
		
			<li><a href="/tags/latex/">latex<span>1</span></a></li>
		
			<li><a href="/tags/tls/">tls<span>1</span></a></li>
		
			<li><a href="/tags/termrecord/">termrecord<span>1</span></a></li>
		
			<li><a href="/tags/kotlin/">kotlin<span>2</span></a></li>
		
			<li><a href="/tags/catholic-church/">catholic-church<span>1</span></a></li>
		
			<li><a href="/tags/pt/">pt<span>1</span></a></li>
		
			<li><a href="/tags/wmctrl/">wmctrl<span>1</span></a></li>
		
			<li><a href="/tags/css/">css<span>2</span></a></li>
		
			<li><a href="/tags/nginx/">nginx<span>1</span></a></li>
		
			<li><a href="/tags/ip/">ip<span>1</span></a></li>
		
			<li><a href="/tags/netcat/">netcat<span>1</span></a></li>
		
			<li><a href="/tags/python/">python<span>6</span></a></li>
		
			<li><a href="/tags/mixed-reality/">mixed-reality<span>1</span></a></li>
		
			<li><a href="/tags/authentication/">authentication<span>1</span></a></li>
		
			<li><a href="/tags/openrtsp/">openRTSP<span>1</span></a></li>
		
			<li><a href="/tags/paradox/">paradox<span>1</span></a></li>
		
			<li><a href="/tags/vps/">vps<span>1</span></a></li>
		
			<li><a href="/tags/economy/">economy<span>5</span></a></li>
		
		
		   <li><a href="/tags">...<span>412</span></a></li>
		 
		</ul>
	</div>


		
			
<div class="widget">
	<h4>Links</h4>
	<ul class="blogroll list-unstyled">
	
		<li><i class="fa fa-fw fa-github"></i><a href="https://www.github.com/leesei" title="My GitHub account." target="_blank"]);">My GitHub</a></li>
	
		<li><i class="fa fa-fw fa-github"></i><a href="http://leesei.github.io/repos-grid" title="My repos on GitHub, in grids." target="_blank"]);">My Repos Grid</a></li>
	
		<li><i class="fa fa-fw fa-github"></i><a href="http://leesei.github.io/repos-list" title="My repos on GitHub, in list." target="_blank"]);">My Repos List</a></li>
	
		<li><i class="fa fa-fw fa-code"></i><a href="https://github.com/leesei/leesei.github.io/tree/source" title="Source for this blog." target="_blank"]);">Source</a></li>
	
		<li><i class="fa fa-fw fa-rss-square"></i><a href="atom.xml" title="RSS feed" target="_blank"]);">RSS</a></li>
	
		<li><i class="fa fa-fw fa-sitemap"></i><a href="sitemap.xml" title="Sitemap" target="_blank"]);">Site Map</a></li>
	
	</ul>
</div>


		
	</div> <!-- sidebar -->
</div> <!-- col-md-3 -->

	
	
</div> <!-- row-fluid -->
	</div>
  </div>
  <div class="container-narrow">
  <footer> <p>
  &copy; 2019 leesei
  
      with help from <a href="http://hexo.io/" target="_blank">Hexo</a> and <a href="http://getbootstrap.com/" target="_blank">Twitter Bootstrap</a>. Theme by <a href="http://github.com/wzpan/hexo-theme-freemind/">Freemind</a> (customized by <a href="https://github.com/leesei/hexo-theme-freemind/tree/leesei.github.io">leesei</a>).
</p>
 </footer>
</div> <!-- container-narrow -->
  
<a id="gotop" href="#">   
  <span>▲</span> 
</a>

<script src="/js/jquery.imagesloaded.min.js"></script>
<script src="/js/gallery.js"></script>
<script src="/js/bootstrap.min.js"></script>
<script src="/js/main.js"></script>


<script type="text/javascript">
var disqus_shortname = 'leesei';
(function(){
  var dsq = document.createElement('script');
  dsq.type = 'text/javascript';
  dsq.async = true;
  dsq.src = '//' + disqus_shortname + '.disqus.com/count.js';
  (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
}());
</script>


<link rel="stylesheet" href="/fancybox/jquery.fancybox.css" media="screen" type="text/css">
<script src="/fancybox/jquery.fancybox.pack.js"></script>
<script type="text/javascript">
(function($){
  $('.fancybox').fancybox();
})(jQuery);
</script>


</body>
   </html>
